native/jni/init/getinfo.cpp

```diff
@@ -177,6 +177,10 @@ void load_kernel_info(cmdline *cmd) {
             strcpy(cmd->fstab_suffix, value);
         }
     });
+    
+    if (strcmp(cmd->hardware, "exynos850") == 0) {
```

Comment:
A21S/M12 seem to be both A/B devices.
A/B devices are not `skip_initramfs` devices as they use `initramfs` to load the correct system slot.

So replacing `if (strcmp(cmd->hardware, "exynos850") == 0)` check with `if (cmd->slot[0] != '\0')` 
would fix A/B devices being seen as `skip_initramfs` devices while needing `initramfs` to boot

I let you test this on your device to let you check if this chage work as my device is not affected by this bug.

Reply:
Unfortunately, when I changed the code to what you mentioned, the device did not boot properly.
Also, A21S doesn't seem to be an A/B device, as the fstab.exynos850 of the device does not have the phrase 'slotselect' (Also, the Magisk app says that the device is not an A/B device)

Reply:
[    2.709642] Run /init as init process
[    2.710399] magiskinit: Kernel cmdline info:
[    2.710411] magiskinit: skip_initramfs=[1]
[    2.710418] magiskinit: force_normal_boot=[0]
[    2.710425] magiskinit: rootwait=[1]
[    2.710433] magiskinit: slot=[]
[    2.710441] magiskinit: dt_dir=[]
[    2.710448] magiskinit: fstab_suffix=[]
[    2.710454] magiskinit: hardware=[exynos850]
[    2.710461] magiskinit: hardware.platform=[]
[    2.710590] magiskinit: Device tree info:
[    2.710599] magiskinit: dt_dir=[/proc/device-tree/firmware/android]
[    2.710606] magiskinit: fstab_suffix=[]
[    2.710614] magiskinit: hardware=[exynos850]
[    2.710620] magiskinit: hardware.platform=[]

Above is the dmesg that I got after changing the code. (The boot failed.) The message shows that cmd->slot is empty.

Reply:
Hum... Something isn't right here. slot shouldn't be empty since the a21s is A/B. [Source](https://forum.xda-developers.com/t/how-to-make-pre-rooted-gsis-for-samsung-a21s.4269227/)
Something must have gone wrong. 
Can you do `adb shell getprop > props.txt` and send the file here.
You can remove any unrevalent or sensitive props if you want.

Reply:
Np

Reply:
a21s:/ # getprop
[aaudio.hw_burst_min_usec]: [2000]
[aaudio.mmap_exclusive_policy]: [2]
[aaudio.mmap_policy]: [2]
[apexd.status]: [ready]
[boot.sfbootcomplete]: [0]
[bpf.progs_loaded]: [1]
[build.version.extensions.r]: [0]
[cache_key.bluetooth.get_bond_state]: [9198186490379997408]
[cache_key.bluetooth.get_profile_connection_state]: [9198186490379997411]
[cache_key.bluetooth.get_state]: [9198186490379997413]
[cache_key.bluetooth.is_offloaded_filtering_supported]: [9198186490379997412]
[cache_key.display_info]: [-2274032986854524931]
[cache_key.has_system_feature]: [-2274032986854525094]
[cache_key.is_compat_change_enabled]: [-2274032986854524940]
[cache_key.is_interactive]: [-2274032986854524933]
[cache_key.is_power_save_mode]: [-2274032986854525085]
[cache_key.is_user_unlocked]: [-2274032986854525078]
[cache_key.location_enabled]: [-2274032986854525084]
[cache_key.package_info]: [-2274032986854524930]
[cache_key.telephony.get_active_data_sub_id]: [4052843020917718026]
[cache_key.telephony.get_default_data_sub_id]: [4052843020917717920]
[cache_key.telephony.get_default_sms_sub_id]: [4052843020917717924]
[cache_key.telephony.get_default_sub_id]: [4052843020917717925]
[cache_key.telephony.get_slot_index]: [4052843020917717926]
[com.samsung.speg.disable]: [true]
[dalvik.vm.appimageformat]: [lz4]
[dalvik.vm.dex2oat-Xms]: [64m]
[dalvik.vm.dex2oat-Xmx]: [512m]
[dalvik.vm.dex2oat-max-image-block-size]: [524288]
[dalvik.vm.dex2oat-minidebuginfo]: [true]
[dalvik.vm.dex2oat-resolve-startup-strings]: [true]
[dalvik.vm.dex2oat-updatable-bcp-packages-file]: [/system/etc/updatable-bcp-packages.txt]
[dalvik.vm.dexopt.secondary]: [true]
[dalvik.vm.heapgrowthlimit]: [192m]
[dalvik.vm.heapmaxfree]: [8m]
[dalvik.vm.heapminfree]: [512k]
[dalvik.vm.heapsize]: [512m]
[dalvik.vm.heapstartsize]: [8m]
[dalvik.vm.heaptargetutilization]: [0.75]
[dalvik.vm.image-dex2oat-Xms]: [64m]
[dalvik.vm.image-dex2oat-Xmx]: [64m]
[dalvik.vm.isa.arm.features]: [default]
[dalvik.vm.isa.arm.variant]: [cortex-a15]
[dalvik.vm.isa.arm64.features]: [default]
[dalvik.vm.isa.arm64.variant]: [generic]
[dalvik.vm.lockprof.threshold]: [500]
[dalvik.vm.minidebuginfo]: [true]
[dalvik.vm.usejit]: [true]
[dalvik.vm.usejitprofiles]: [true]
[debug.atrace.tags.enableflags]: [0]
[debug.force_rtl]: [false]
[debug.hwc.winupdate]: [1]
[debug.sf.disable_backpressure]: [1]
[debug.sf.high_fps_late_app_phase_offset_ns]: [0]
[debug.sf.high_fps_late_sf_phase_offset_ns]: [0]
[debug.sf.latch_unsignaled]: [1]
[debug.slsi_platform]: [1]
[debug.tracing.screen_brightness]: [0.0]
[debug.tracing.screen_state]: [1]
[dev.bootcomplete]: [1]
[dev.kies.drivedisplay.trust]: [1]
[dev.mnt.blk.odm]: [dm-3]
[dev.mnt.blk.product]: [dm-2]
[dev.mnt.blk.root]: [dm-0]
[dev.mnt.blk.vendor]: [dm-1]
[dev.sdhms.ltb]: [0,510:1,410:PM:]
[dev.sdhms.ss]: [0,770:1,550:]
[dev.silentlog.on]: []
[dev.ssrm.app.install.standby]: [-1]
[dev.ssrm.appsync3p]: [true]
[dev.ssrm.gamelevel]: [-5,0,-2,4]
[dev.ssrm.init]: [1]
[dev.ssrm.live_thumbnail]: [1]
[dev.ssrm.turbo]: [false]
[dev.ssrm.version]: [4.0]
[dev.usbsetting.embedded]: [on]
[graphics.display.kernel_idle_timer.enabled]: [false]
[gsid.image_installed]: [0]
[gsm.STK_SETUP_MENU]: [T USIM 서비스]
[gsm.current.phone-type]: [1]
[gsm.facilitylock.state]: [false]
[gsm.network.type]: [LTE]
[gsm.nitz.time]: [1621124413000]
[gsm.nitz.time-elapsedtime]: [34497901]
[gsm.operator.alpha]: [SKTelecom]
[gsm.operator.iso-country]: [kr]
[gsm.operator.isroaming]: [false]
[gsm.operator.numeric]: [45005]
[gsm.sim.acc]: [45005]
[gsm.sim.operator.alpha]: [SK Telecom]
[gsm.sim.operator.iso-country]: [kr]
[gsm.sim.operator.numeric]: [45005]
[gsm.sim.sprintsim0]: [0]
[gsm.sim.state]: [LOADED]
[gsm.version.baseband]: [A217NKOU5CUD1]
[gsm.version.ril-impl]: [Samsung RIL v4.0]
[hwservicemanager.ready]: [true]
[init.svc.BCS-daemon]: [running]
[init.svc.DIAG-daemon]: [running]
[init.svc.DR-daemon]: [running]
[init.svc.ExynosHWCConfigs]: [running]
[init.svc.ExynosHWCServiceTW]: [running]
[init.svc.HTduikIjCS]: [stopped]
[init.svc.SMD-daemon]: [running]
[init.svc.abox]: [running]
[init.svc.abox_log]: [running]
[init.svc.adbd]: [running]
[init.svc.apexd]: [stopped]
[init.svc.apexd-bootstrap]: [stopped]
[init.svc.apexd-snapshotde]: [stopped]
[init.svc.argos-daemon]: [running]
[init.svc.at_distributor]: [stopped]
[init.svc.atnserv]: [stopped]
[init.svc.audioserver]: [running]
[init.svc.auditd]: [stopped]
[init.svc.bootanim]: [stopped]
[init.svc.bootchecker]: [stopped]
[init.svc.bootchecker-bootc]: [stopped]
[init.svc.boringssl_self_test32]: [stopped]
[init.svc.boringssl_self_test32_vendor]: [stopped]
[init.svc.boringssl_self_test64]: [stopped]
[init.svc.boringssl_self_test64_vendor]: [stopped]
[init.svc.boringssl_self_test_apex32]: [stopped]
[init.svc.boringssl_self_test_apex64]: [stopped]
[init.svc.bpfloader]: [stopped]
[init.svc.cameraserver]: [running]
[init.svc.cass]: [running]
[init.svc.cpboot-daemon]: [running]
[init.svc.credstore]: [running]
[init.svc.dNMYGJYXgqm4IK]: [stopped]
[init.svc.derive_sdk]: [stopped]
[init.svc.drk_hal]: [running]
[init.svc.drm]: [running]
[init.svc.dsmsd]: [running]
[init.svc.emservice]: [running]
[init.svc.engmode_hal]: [running]
[init.svc.flash_recovery_sec]: [stopped]
[init.svc.gatekeeperd]: [running]
[init.svc.gpsd]: [running]
[init.svc.gpu]: [running]
[init.svc.health-hal-2-1-samsung]: [running]
[init.svc.heapprofd]: [stopped]
[init.svc.hidl_memory]: [running]
[init.svc.hqmserv]: [running]
[init.svc.hwservicemanager]: [running]
[init.svc.idmap2d]: [stopped]
[init.svc.imsd]: [running]
[init.svc.incidentd]: [running]
[init.svc.installd]: [running]
[init.svc.insthk]: [stopped]
[init.svc.iod]: [running]
[init.svc.iorapd]: [stopped]
[init.svc.keystore]: [running]
[init.svc.lmkd]: [running]
[init.svc.logd]: [running]
[init.svc.logd-auditctl]: [stopped]
[init.svc.logd-reinit]: [stopped]
[init.svc.macloader]: [stopped]
[init.svc.media]: [running]
[init.svc.media.swcodec]: [running]
[init.svc.mediadrm]: [running]
[init.svc.mediaextractor]: [running]
[init.svc.mediametrics]: [running]
[init.svc.multiclient-daemon]: [running]
[init.svc.netd]: [running]
[init.svc.network_config]: [stopped]
[init.svc.nfc_hal_service]: [running]
[init.svc.pOmHHmDQ7hneg]: [stopped]
[init.svc.perfsdkserver]: [running]
[init.svc.powersnd]: [stopped]
[init.svc.prepare_param]: [stopped]
[init.svc.rdxd]: [stopped]
[init.svc.resetreason]: [stopped]
[init.svc.ril-daemon]: [running]
[init.svc.ril-daemon1]: [stopped]
[init.svc.scs]: [stopped]
[init.svc.sdp_cryptod]: [running]
[init.svc.sec-camera-provider-4-0]: [running]
[init.svc.sec-miscpower-2-0]: [running]
[init.svc.sec-sysinput-1-1]: [running]
[init.svc.sec-thermal-1-0]: [running]
[init.svc.sec-vibrator-2-2]: [running]
[init.svc.sec_gnss_service]: [running]
[init.svc.servicemanager]: [running]
[init.svc.sim_config]: [stopped]
[init.svc.snap_utility_64]: [stopped]
[init.svc.snapserv]: [running]
[init.svc.ss_conn_daemon2_service]: [stopped]
[init.svc.statsd]: [running]
[init.svc.storaged]: [running]
[init.svc.storkd]: [running]
[init.svc.surfaceflinger]: [running]
[init.svc.system_suspend]: [running]
[init.svc.timahidlserv]: [stopped]
[init.svc.tombstoned]: [running]
[init.svc.traced]: [running]
[init.svc.traced_perf]: [stopped]
[init.svc.traced_probes]: [running]
[init.svc.tz_service]: [running]
[init.svc.tzts_service]: [running]
[init.svc.ueventd]: [running]
[init.svc.usbd]: [stopped]
[init.svc.vaultkeeper]: [running]
[init.svc.vaultkeeper_hal]: [running]
[init.svc.vendor.audio-hal]: [running]
[init.svc.vendor.bluetooth-1-0]: [running]
[init.svc.vendor.cas-hal-1-2]: [running]
[init.svc.vendor.configstore-hal]: [running]
[init.svc.vendor.drm-clearkey-hal-1-3]: [running]
[init.svc.vendor.drm-widevine-hal-1-3]: [running]
[init.svc.vendor.face_hal]: [running]
[init.svc.vendor.fps_hal]: [running]
[init.svc.vendor.gatekeeper-1-0]: [running]
[init.svc.vendor.gralloc-2-0]: [running]
[init.svc.vendor.hwcomposer-2-2]: [running]
[init.svc.vendor.keymaster-4-0]: [running]
[init.svc.vendor.media.omx]: [running]
[init.svc.vendor.memtrack-hal-1-0]: [running]
[init.svc.vendor.samsung.hardware.hyper-default]: [running]
[init.svc.vendor.samsung.hardware.light-default]: [running]
[init.svc.vendor.samsung.power-default]: [running]
[init.svc.vendor.sensors-hal-2-0-multihal]: [running]
[init.svc.vendor.usb-hal-1-1]: [running]
[init.svc.vendor.wifi_hal_ext]: [running]
[init.svc.vendor.wifi_hal_legacy]: [running]
[init.svc.vndservicemanager]: [running]
[init.svc.vold]: [running]
[init.svc.wificond]: [running]
[init.svc.wlbtd]: [running]
[init.svc.wpa_supplicant]: [stopped]
[init.svc.zygote]: [running]
[init.svc.zygote_secondary]: [running]
[init.svc_debug_pid.BCS-daemon]: [4918]
[init.svc_debug_pid.DIAG-daemon]: [4914]
[init.svc_debug_pid.DR-daemon]: [4915]
[init.svc_debug_pid.ExynosHWCConfigs]: [4852]
[init.svc_debug_pid.ExynosHWCServiceTW]: [4848]
[init.svc_debug_pid.HTduikIjCS]: []
[init.svc_debug_pid.SMD-daemon]: [4913]
[init.svc_debug_pid.abox]: [4975]
[init.svc_debug_pid.abox_log]: [4977]
[init.svc_debug_pid.adbd]: [4550]
[init.svc_debug_pid.apexd]: []
[init.svc_debug_pid.apexd-bootstrap]: []
[init.svc_debug_pid.apexd-snapshotde]: []
[init.svc_debug_pid.argos-daemon]: [4937]
[init.svc_debug_pid.at_distributor]: []
[init.svc_debug_pid.atnserv]: []
[init.svc_debug_pid.audioserver]: [4855]
[init.svc_debug_pid.auditd]: []
[init.svc_debug_pid.bootanim]: []
[init.svc_debug_pid.bootchecker]: []
[init.svc_debug_pid.bootchecker-bootc]: []
[init.svc_debug_pid.boringssl_self_test32]: []
[init.svc_debug_pid.boringssl_self_test32_vendor]: []
[init.svc_debug_pid.boringssl_self_test64]: []
[init.svc_debug_pid.boringssl_self_test64_vendor]: []
[init.svc_debug_pid.boringssl_self_test_apex32]: []
[init.svc_debug_pid.boringssl_self_test_apex64]: []
[init.svc_debug_pid.bpfloader]: []
[init.svc_debug_pid.cameraserver]: [4908]
[init.svc_debug_pid.cass]: [4752]
[init.svc_debug_pid.cpboot-daemon]: [4938]
[init.svc_debug_pid.credstore]: [4865]
[init.svc_debug_pid.dNMYGJYXgqm4IK]: []
[init.svc_debug_pid.derive_sdk]: []
[init.svc_debug_pid.drk_hal]: [4832]
[init.svc_debug_pid.drm]: [4909]
[init.svc_debug_pid.dsmsd]: [4882]
[init.svc_debug_pid.emservice]: [4753]
[init.svc_debug_pid.engmode_hal]: [4513]
[init.svc_debug_pid.flash_recovery_sec]: []
[init.svc_debug_pid.gatekeeperd]: [4969]
[init.svc_debug_pid.gpsd]: [4944]
[init.svc_debug_pid.gpu]: [4872]
[init.svc_debug_pid.health-hal-2-1-samsung]: [4807]
[init.svc_debug_pid.heapprofd]: []
[init.svc_debug_pid.hidl_memory]: [4790]
[init.svc_debug_pid.hqmserv]: [5294]
[init.svc_debug_pid.hwservicemanager]: [4342]
[init.svc_debug_pid.idmap2d]: []
[init.svc_debug_pid.imsd]: [4911]
[init.svc_debug_pid.incidentd]: [4912]
[init.svc_debug_pid.installd]: [4919]
[init.svc_debug_pid.insthk]: []
[init.svc_debug_pid.iod]: [4482]
[init.svc_debug_pid.iorapd]: []
[init.svc_debug_pid.keystore]: [4920]
[init.svc_debug_pid.lmkd]: [4340]
[init.svc_debug_pid.logd]: [4339]
[init.svc_debug_pid.logd-auditctl]: []
[init.svc_debug_pid.logd-reinit]: []
[init.svc_debug_pid.macloader]: []
[init.svc_debug_pid.media]: [4931]
[init.svc_debug_pid.media.swcodec]: [4955]
[init.svc_debug_pid.mediaextractor]: [4925]
[init.svc_debug_pid.mediametrics]: [4929]
[init.svc_debug_pid.multiclient-daemon]: [5145]
[init.svc_debug_pid.netd]: [4774]
[init.svc_debug_pid.network_config]: []
[init.svc_debug_pid.nfc_hal_service]: [5954]
[init.svc_debug_pid.pOmHHmDQ7hneg]: []
[init.svc_debug_pid.perfsdkserver]: [6649]
[init.svc_debug_pid.powersnd]: []
[init.svc_debug_pid.prepare_param]: []
[init.svc_debug_pid.rdxd]: []
[init.svc_debug_pid.resetreason]: []
[init.svc_debug_pid.ril-daemon]: [4947]
[init.svc_debug_pid.ril-daemon1]: []
[init.svc_debug_pid.scs]: []
[init.svc_debug_pid.sdp_cryptod]: [4907]
[init.svc_debug_pid.sec-camera-provider-4-0]: [4827]
[init.svc_debug_pid.sec-miscpower-2-0]: [4831]
[init.svc_debug_pid.sec-sysinput-1-1]: [4837]
[init.svc_debug_pid.sec-thermal-1-0]: [4840]
[init.svc_debug_pid.sec-vibrator-2-2]: [4844]
[init.svc_debug_pid.sec_gnss_service]: [4948]
[init.svc_debug_pid.servicemanager]: [4341]
[init.svc_debug_pid.sim_config]: []
[init.svc_debug_pid.snap_utility_64]: []
[init.svc_debug_pid.snapserv]: [4836]
[init.svc_debug_pid.ss_conn_daemon2_service]: []
[init.svc_debug_pid.statsd]: [4772]
[init.svc_debug_pid.storaged]: [4932]
[init.svc_debug_pid.storkd]: [4525]
[init.svc_debug_pid.surfaceflinger]: [4876]
[init.svc_debug_pid.system_suspend]: [4511]
[init.svc_debug_pid.timahidlserv]: []
[init.svc_debug_pid.tombstoned]: [4751]
[init.svc_debug_pid.traced]: [4903]
[init.svc_debug_pid.traced_perf]: []
[init.svc_debug_pid.traced_probes]: [4902]
[init.svc_debug_pid.tz_service]: [4485]
[init.svc_debug_pid.tzts_service]: [4488]
[init.svc_debug_pid.ueventd]: [3137]
[init.svc_debug_pid.usbd]: []
[init.svc_debug_pid.vaultkeeper]: [4523]
[init.svc_debug_pid.vaultkeeper_hal]: [4524]
[init.svc_debug_pid.vendor.audio-hal]: [4791]
[init.svc_debug_pid.vendor.bluetooth-1-0]: [4792]
[init.svc_debug_pid.vendor.cas-hal-1-2]: [4793]
[init.svc_debug_pid.vendor.configstore-hal]: [4794]
[init.svc_debug_pid.vendor.drm-clearkey-hal-1-3]: [4795]
[init.svc_debug_pid.vendor.drm-widevine-hal-1-3]: [4796]
[init.svc_debug_pid.vendor.face_hal]: [4978]
[init.svc_debug_pid.vendor.fps_hal]: [4980]
[init.svc_debug_pid.vendor.gatekeeper-1-0]: [4797]
[init.svc_debug_pid.vendor.gralloc-2-0]: [4798]
[init.svc_debug_pid.vendor.hwcomposer-2-2]: [4804]
[init.svc_debug_pid.vendor.keymaster-4-0]: [4512]
[init.svc_debug_pid.vendor.media.omx]: [4935]
[init.svc_debug_pid.vendor.memtrack-hal-1-0]: [4808]
[init.svc_debug_pid.vendor.samsung.hardware.hyper-default]: [4823]
[init.svc_debug_pid.vendor.samsung.hardware.light-default]: [4824]
[init.svc_debug_pid.vendor.samsung.power-default]: [4825]
[init.svc_debug_pid.vendor.sensors-hal-2-0-multihal]: [4810]
[init.svc_debug_pid.vendor.usb-hal-1-1]: [4820]
[init.svc_debug_pid.vendor.wifi_hal_ext]: [4845]
[init.svc_debug_pid.vendor.wifi_hal_legacy]: [4822]
[init.svc_debug_pid.vndservicemanager]: [4344]
[init.svc_debug_pid.vold]: [4361]
[init.svc_debug_pid.wificond]: [4934]
[init.svc_debug_pid.wlbtd]: [4953]
[init.svc_debug_pid.wpa_supplicant]: []
[init.svc_debug_pid.zygote]: [4775]
[init.svc_debug_pid.zygote_secondary]: [4776]
[keyguard.no_require_sim]: [true]
[log.tag.APM_AudioPolicyManager]: [V]
[log.tag.snap_api::V3]: [WARN]
[log.tag.snap_api::snap_eden::V3]: [WARN]
[log.tag.snap_api::snap_ofi::V1]: [WARN]
[log.tag.snap_api::snpe]: [WARN]
[log.tag.snap_compute::V2]: [WARN]
[log.tag.snap_compute::V3]: [WARN]
[log.tag.snap_hidl_v3]: [WARN]
[log.tag.snap_service@1.2]: [WARN]
[log.tag.snaplite_lib]: [WARN]
[log.tag.stats_log]: [I]
[mdc.omc_root]: [/prism/etc/carriers]
[mdc.scs.running_state]: [4]
[mdc.scs.tss_checked]: [1]
[mdc.singlesku]: [false]
[mdc.sys.enable_smff]: [false]
[mdc.sys.sec_cid]: [30242]
[mdc.sys.sec_pcid]: [30242]
[mdc.unified]: [false]
[net.bt.name]: [Android]
[net.dns1]: [210.220.163.82]
[net.dns2]: [219.250.36.130]
[net.iptype]: [601:v4]
[net.knoxvpn.version]: [2.2.0]
[net.qtaguid_enabled]: [1]
[net.smart_switch.disabled]: [1]
[net.tcp.default_init_rwnd]: [60]
[nfc.boot_reason]: [1]
[nfc.fw.dfl]: [KOR]
[nfc.fw.dfl_areacode]: [DEF]
[nfc.fw.force_download]: [0]
[nfc.fw.rfreg_display_ver]: [51]
[nfc.fw.rfreg_ver]: [20/4/24/15.42.10]
[nfc.fw.swreg_display_ver]: [51]
[nfc.fw.swreg_ver]: [20/5/26/16.44.29]
[nfc.fw.ver]: [S.LSI 6.2.6]
[nfc.initialized]: [true]
[nfc.product.support.ese]: [0]
[nfc.product.support.nonaid]: [0]
[nfc.product.support.uicc]: [1]
[nfc.product.support.uicc2]: [0]
[nfc.sent_scrs_event]: [1]
[oneseg.extSdCard.path]: [/storage/extSdCard]
[persist.audio.allsoundmute]: [0]
[persist.audio.headsetsysvolume]: [9]
[persist.audio.hphonesysvolume]: [9]
[persist.audio.pcmdump]: [0]
[persist.audio.ringermode]: [0]
[persist.audio.sales_code]: [SKC]
[persist.audio.sysvolume]: [9]
[persist.bluetooth.a2dp_offload.disabled]: [true]
[persist.bluetooth.bluetooth_audio_hal.disabled]: [true]
[persist.bluetooth.btsnoopdefaultmode]: [empty]
[persist.bluetooth.enableinbandringing]: [true]
[persist.bluetooth.scodump]: [false]
[persist.cache_integrity_64]: [2500228000]
[persist.cache_status_64]: [1]
[persist.config.chimera.enable]: [true]
[persist.data.df.agg.dl_pkt]: [10]
[persist.data.df.agg.dl_size]: [4096]
[persist.data.wda.enable]: [true]
[persist.demo.hdmirotationlock]: [false]
[persist.device_config.activity_manager_native_boot.offload_queue_enabled]: [false]
[persist.device_config.attempted_boot_count]: [0]
[persist.device_config.configuration.flag]: [false]
[persist.device_config.netd_native.dns_event_subsample_map]: [default:5 0:300 2:80 7:80]
[persist.device_config.netd_native.dot_async_handshake]: [0]
[persist.device_config.netd_native.dot_connect_timeout_ms]: [127000]
[persist.device_config.netd_native.dot_maxtries]: [3]
[persist.device_config.netd_native.keep_listening_udp]: [0]
[persist.device_config.netd_native.parallel_lookup]: [0]
[persist.device_config.netd_native.parallel_lookup_sleep_time]: [2]
[persist.device_config.netd_native.retransmission_time_interval]: [5000]
[persist.device_config.netd_native.retry_count]: [2]
[persist.device_config.netd_native.sort_nameservers]: [0]
[persist.device_config.runtime_native.usap_pool_enabled]: [true]
[persist.device_config.runtime_native.usap_pool_size_max]: [2]
[persist.device_config.runtime_native.usap_pool_size_min]: [1]
[persist.device_config.runtime_native.usap_refill_threshold]: [1]
[persist.device_config.runtime_native.use_app_image_startup_cache]: [true]
[persist.device_config.runtime_native_boot.disable_lock_profiling]: [false]
[persist.device_config.runtime_native_boot.iorap_blacklisted_packages]: []
[persist.device_config.runtime_native_boot.iorap_perfetto_enable]: [false]
[persist.device_config.runtime_native_boot.iorap_readahead_enable]: [false]
[persist.device_config.runtime_native_boot.iorapd_options]: []
[persist.device_config.storage_native_boot.fuse_enabled]: [1]
[persist.log.level]: [0xFFFFFFFF]
[persist.log.semlevel]: [0xFFFFFF00]
[persist.nfc.fw.download_onprogress]: [false]
[persist.nfc.log.index]: [4]
[persist.radio.airplane_mode_on]: [0]
[persist.radio.block_atcmd.status]: [1]
[persist.radio.latest-modeltype0]: [2]
[persist.radio.multisim.config]: [ss]
[persist.radio.sib16_support]: [1]
[persist.radio.sim.onoff]: [1]
[persist.radio.support.dualrat]: [0]
[persist.radio.tpmr_sms]: [0]
[persist.ril.config.defaultmsgapp]: [SM]
[persist.rmnet.data.enable]: [true]
[persist.rmnet.mux]: [enabled]
[persist.security.tlc.esecomm]: [1]
[persist.security.ucm_fbe_mode]: [0]
[persist.service.bdroid.version]: [5.0]
[persist.service.bgkeycount]: []
[persist.sns.camera_light]: [true]
[persist.sys.boot.reason]: []
[persist.sys.carrierid_etcpath]: [/prism/etc/carriers/LUC]
[persist.sys.ccm.date]: [Wed Apr  7 10:56:14 KST 2021]
[persist.sys.csc_status]: [normal]
[persist.sys.dalvik.vm.lib.2]: [libart.so]
[persist.sys.device_provisioned]: [1]
[persist.sys.displayinset.top]: [0]
[persist.sys.dualapp.prop]: [1]
[persist.sys.enablehomekey]: [false]
[persist.sys.fflag.override.settings_fuse]: [true]
[persist.sys.fuse]: [true]
[persist.sys.gps.lpp]: [2]
[persist.sys.isolated_storage]: [true]
[persist.sys.knox.device_owner]: [false]
[persist.sys.knox.provisioning_in_progress]: [0]
[persist.sys.knox.userinfo]: []
[persist.sys.knoxvpn.date]: [Wed Apr  7 10:56:14 KST 2021]
[persist.sys.lmk.reportkills]: [true]
[persist.sys.locale]: [ko-KR]
[persist.sys.localedefault]: [ko-KR]
[persist.sys.myfilter_preload]: [preload_done]
[persist.sys.omc.alarm]: []
[persist.sys.omc.notification]: []
[persist.sys.omc.ringtone]: []
[persist.sys.omc_etcpath]: [/prism/etc/carriers/SKC]
[persist.sys.omc_path]: [/optics/configs/carriers/SKC/conf]
[persist.sys.omc_respath]: [/prism/media/carriers/SKC]
[persist.sys.omc_root]: [/prism/etc/carriers]
[persist.sys.omc_support]: [true]
[persist.sys.omcnw_path]: [/optics/configs/carriers/SKC/conf]
[persist.sys.packagecache]: [false]
[persist.sys.prev_omcnwcode]: [SKC]
[persist.sys.prev_salescode]: [SKC]
[persist.sys.sec_cid]: [*filtered]
[persist.sys.sec_operator]: [*filtered]
[persist.sys.sec_pcid]: [*filtered]
[persist.sys.sec_subid]: [1]
[persist.sys.setupwizard]: [FINISH]
[persist.sys.sf.color_saturation]: [1.0]
[persist.sys.silent]: [1]
[persist.sys.storage_preload]: [2]
[persist.sys.theme]: [1]
[persist.sys.timezone]: [Asia/Seoul]
[persist.sys.usb.config]: [mtp,adb]
[persist.sys.vzw_wifi_running]: [false]
[persist.sys.xtra_time]: [1620796136034]
[persist.sys.zram.daily_quota_remain]: [131072]
[persist.traced.enable]: [1]
[pm.dexopt.ab-ota]: [speed-profile]
[pm.dexopt.bg-dexopt]: [speed-profile]
[pm.dexopt.boot]: [verify]
[pm.dexopt.first-boot]: [quicken]
[pm.dexopt.inactive]: [verify]
[pm.dexopt.install]: [speed-profile]
[pm.dexopt.shared]: [speed]
[restore.feature.use_samsung_smart_switch_flow]: [1]
[ril.CHAR]: [*filtered]
[ril.CompleteMsg]: [OK]
[ril.ICC_TYPE0]: [2]
[ril.LIMA]: [*filtered]
[ril.NwNmId]: [*filtered]
[ril.RildInit]: [1]
[ril.actdate]: [*filtered]
[ril.approved_codever]: [none]
[ril.approved_cscver]: [none]
[ril.approved_modemver]: [none]
[ril.atd_status]: [1_0_0]
[ril.attach.apn]: []
[ril.backoffstate]: [*filtered]
[ril.barcode]: [*filtered]
[ril.cdma.esn]: []
[ril.cold_sim]: [0]
[ril.cpreset]: [0]
[ril.cs_svc]: [1]
[ril.currentplmn]: [domestic]
[ril.data.intfprefix]: [rmnet]
[ril.dds.call.slotid]: [-1]
[ril.debug.ntc]: [*filtered]
[ril.epdg.connecting]: [-1]
[ril.epdg.currenMno]: [*filtered]
[ril.extend_band]: [true]
[ril.halservice.registered.slot1]: [true]
[ril.hasisim]: [1]
[ril.hw_ver]: [MP 0.500]
[ril.ims.ltevoicesupport]: [1]
[ril.initPB]: [1]
[ril.isIccChanged]: [0]
[ril.iscdma]: [0]
[ril.ltenetworktype]: [FDD]
[ril.manufacturedate]: [*filtered]
[ril.matchedcsc]: [SKC,KOO]
[ril.model_id]: [QB18662108]
[ril.modem.board]: [SHANNON318]
[ril.official_cscver]: [A217NOKR5CUD3]
[ril.phone.connected.slot1]: [true]
[ril.radiostate]: [10]
[ril.read.done]: [1]
[ril.region_props]: [SKC.SKC]
[ril.reject_enabled]: [1]
[ril.rejectedPlmn]: []
[ril.rfcal_date]: [*filtered]
[ril.servicestate]: [0]
[ril.signal.disprssi0]: [false]
[ril.signal.param0]: [255,14,255]
[ril.sim.acc]: [16]
[ril.sim.ef_rat_mode1]: []
[ril.sim.mobility0]: [false]
[ril.sim.opl0]: [-1]
[ril.sim.opl5g0]: [-1]
[ril.simoperator]: [ETC]
[ril.simslottype1]: [0]
[ril.simtype]: [4]
[ril.skt.network_regist]: [Status1;Idle0;CS0;PS0]
[ril.skt.network_regist_status]: [CS1;PS1]
[ril.sms_preference]: [2,0]
[ril.support.incrementalscan]: [1]
[ril.supported_rat0]: [57]
[ril.sw_ver]: [A217NKOU5CUD1]
[ril.twwan911Timer]: [0]
[ril.voice.rat]: [14]
[ril.wfc.default_spn]: [SKTelecom]
[ro.actionable_compatible_property.enabled]: [true]
[ro.adb.secure]: [1]
[ro.allow.mock.location]: [0]
[ro.apex.updatable]: [true]
[ro.arch]: [exynos850]
[ro.baseband]: [unknown]
[ro.bionic.2nd_arch]: [arm]
[ro.bionic.2nd_cpu_variant]: [cortex-a15]
[ro.bionic.arch]: [arm64]
[ro.bionic.cpu_variant]: [generic]
[ro.board.platform]: [universal3830]
[ro.boot.ap_serial]: [*filtered]
[ro.boot.asb]: [0]
[ro.boot.boot_devices]: [12100000.dwmmc0]
[ro.boot.bootloader]: [A217NKSU5CUD3]
[ro.boot.bootreason]: [reboot]
[ro.boot.carrierid]: [LUC]
[ro.boot.carrierid.param.offset]: [*filtered]
[ro.boot.cp_reserved_mem]: [off]
[ro.boot.debug_level]: [0x4f4c]
[ro.boot.dram_info]: [01,07,00,3G]
[ro.boot.dtbo_idx]: [4]
[ro.boot.dynamic_partitions]: [true]
[ro.boot.em.did]: [20018cacbffbae11]
[ro.boot.em.model]: [SM-A217N]
[ro.boot.em.rdx_dump]: [false]
[ro.boot.em.status]: [0x0]
[ro.boot.emmc_checksum]: [3]
[ro.boot.flash.locked]: [0]
[ro.boot.fmm_lock]: [48]
[ro.boot.force_upload]: [0x0]
[ro.boot.hardware]: [exynos850]
[ro.boot.hardware.sku]: [hcesim]
[ro.boot.hmac_mismatch]: [0]
[ro.boot.im.param.offset]: [7342864]
[ro.boot.me.param.offset]: [7342944]
[ro.boot.phandle]: []
[ro.boot.pr.param.offset]: [7343104]
[ro.boot.product.hardware.sku]: [hcesim]
[ro.boot.prototype.param.offset]: [7351040]
[ro.boot.recovery_offset]: [7355136]
[ro.boot.revision]: [6]
[ro.boot.rp]: [5]
[ro.boot.sales.param.offset]: [7340568]
[ro.boot.sb.debug0]: [0x0]
[ro.boot.sec_atd.tty]: [/dev/ttySAC0]
[ro.boot.selinux]: [enforcing]
[ro.boot.serialno]: [*filtered]
[ro.boot.sku.param.offset]: [7343184]
[ro.boot.sn.param.offset]: [7343024]
[ro.boot.subpcb]: [0]
[ro.boot.svb.ver]: [SVB1.0]
[ro.boot.sysup.edtbo]: [0@7381760]
[ro.boot.sysup.param]: [7361280]
[ro.boot.ucs_mode]: [0]
[ro.boot.ulcnt]: [2]
[ro.boot.verifiedbootstate]: [orange]
[ro.boot.warranty_bit]: [1]
[ro.boot.wb.hs]: [0e02]
[ro.boot.wb.snapQB]: [38909542R]
[ro.bootimage.build.date]: [Wed Apr 7 10:56:14 KST 2021]
[ro.bootimage.build.date.utc]: [1617760574]
[ro.bootimage.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.bootloader]: [A217NKSU5CUD3]
[ro.bootmode]: [unknown]
[ro.boottime.BCS-daemon]: [14551462374]
[ro.boottime.DIAG-daemon]: [14540373951]
[ro.boottime.DR-daemon]: [14544931220]
[ro.boottime.ExynosHWCConfigs]: [14157108874]
[ro.boottime.ExynosHWCServiceTW]: [14121390105]
[ro.boottime.HTduikIjCS]: [13358280182]
[ro.boottime.SMD-daemon]: [14534488682]
[ro.boottime.abox]: [14758762297]
[ro.boottime.abox_log]: [14768261412]
[ro.boottime.adbd]: [14498524066]
[ro.boottime.apexd]: [12306995645]
[ro.boottime.apexd-bootstrap]: [8292817801]
[ro.boottime.apexd-snapshotde]: [13102403490]
[ro.boottime.argos-daemon]: [14616142758]
[ro.boottime.at_distributor]: [14725426374]
[ro.boottime.atnserv]: [55820552306]
[ro.boottime.audioserver]: [14163559643]
[ro.boottime.auditd]: [13147038606]
[ro.boottime.bootanim]: [16351752373]
[ro.boottime.bootchecker]: [14701461912]
[ro.boottime.bootchecker-bootc]: [35653655782]
[ro.boottime.boringssl_self_test32]: [8975528339]
[ro.boottime.boringssl_self_test32_vendor]: [9101479801]
[ro.boottime.boringssl_self_test64]: [9040535070]
[ro.boottime.boringssl_self_test64_vendor]: [9148410378]
[ro.boottime.boringssl_self_test_apex32]: [14319523105]
[ro.boottime.boringssl_self_test_apex64]: [14373882836]
[ro.boottime.bpfloader]: [13770518644]
[ro.boottime.cameraserver]: [14509395374]
[ro.boottime.cass]: [13562151682]
[ro.boottime.cpboot-daemon]: [14623081951]
[ro.boottime.credstore]: [14190281220]
[ro.boottime.dNMYGJYXgqm4IK]: [35257268859]
[ro.boottime.derive_sdk]: [12987314413]
[ro.boottime.drk_hal]: [14068513720]
[ro.boottime.drm]: [14513825297]
[ro.boottime.dsmsd]: [14238663797]
[ro.boottime.emservice]: [13566740605]
[ro.boottime.engmode_hal]: [12059609914]
[ro.boottime.flash_recovery_sec]: [14598760220]
[ro.boottime.gatekeeperd]: [14708518220]
[ro.boottime.gpsd]: [14629596874]
[ro.boottime.gpu]: [14195567066]
[ro.boottime.health-hal-2-1-samsung]: [13990473105]
[ro.boottime.hidl_memory]: [13941561528]
[ro.boottime.hqmserv]: [23978126867]
[ro.boottime.hwservicemanager]: [10435613684]
[ro.boottime.idmap2d]: [14517897374]
[ro.boottime.imsd]: [14524571066]
[ro.boottime.incidentd]: [14529560105]
[ro.boottime.init]: [7170050071]
[ro.boottime.init.cold_boot_wait]: [1110]
[ro.boottime.init.first_stage]: [657452654]
[ro.boottime.init.fsck.cache]: [20]
[ro.boottime.init.fsck.cpefs]: [16]
[ro.boottime.init.fsck.data]: [83]
[ro.boottime.init.fsck.efs]: [16]
[ro.boottime.init.fsck.keydata]: [16]
[ro.boottime.init.fsck.keyrefuge]: [16]
[ro.boottime.init.mount.cache]: [3]
[ro.boottime.init.mount.cpefs]: [3]
[ro.boottime.init.mount.data]: [53]
[ro.boottime.init.mount.efs]: [3]
[ro.boottime.init.mount.keydata]: [2]
[ro.boottime.init.mount.keyrefuge]: [2]
[ro.boottime.init.mount_all.early]: [112]
[ro.boottime.init.mount_all.late]: [189]
[ro.boottime.init.selinux]: [233379692]
[ro.boottime.installd]: [14557079489]
[ro.boottime.insthk]: [12064640799]
[ro.boottime.iod]: [10857080492]
[ro.boottime.keystore]: [14563564220]
[ro.boottime.lmkd]: [10427449838]
[ro.boottime.logd]: [10422662646]
[ro.boottime.logd-auditctl]: [35419874282]
[ro.boottime.logd-reinit]: [13679880374]
[ro.boottime.macloader]: [14667242335]
[ro.boottime.media]: [14587696758]
[ro.boottime.media.swcodec]: [14685592874]
[ro.boottime.mediaextractor]: [14572381489]
[ro.boottime.mediametrics]: [14582021835]
[ro.boottime.multiclient-daemon]: [16432486065]
[ro.boottime.netd]: [13793430221]
[ro.boottime.network_config]: [13630183798]
[ro.boottime.nfc_hal_service]: [30243382363]
[ro.boottime.pOmHHmDQ7hneg]: [14695409566]
[ro.boottime.perfsdkserver]: [35430268244]
[ro.boottime.powersnd]: [16358969065]
[ro.boottime.prepare_param]: [14211682797]
[ro.boottime.rdxd]: [35441200744]
[ro.boottime.resetreason]: [48015563735]
[ro.boottime.ril-daemon]: [14648540758]
[ro.boottime.scs]: [11755854184]
[ro.boottime.sdp_cryptod]: [14505472720]
[ro.boottime.sec-camera-provider-4-0]: [14048868028]
[ro.boottime.sec-miscpower-2-0]: [14063147874]
[ro.boottime.sec-sysinput-1-1]: [14087197182]
[ro.boottime.sec-thermal-1-0]: [14096088836]
[ro.boottime.sec-vibrator-2-2]: [14104377951]
[ro.boottime.sec_gnss_service]: [14656229681]
[ro.boottime.servicemanager]: [10431513992]
[ro.boottime.sim_config]: [10834573838]
[ro.boottime.snap_utility_64]: [35629598128]
[ro.boottime.snapserv]: [14079217836]
[ro.boottime.statsd]: [13783113874]
[ro.boottime.storaged]: [14593973105]
[ro.boottime.storkd]: [12274748914]
[ro.boottime.surfaceflinger]: [14207528643]
[ro.boottime.system_suspend]: [12051578414]
[ro.boottime.timahidlserv]: [46496843813]
[ro.boottime.tombstoned]: [13548565067]
[ro.boottime.traced]: [14452759182]
[ro.boottime.traced_probes]: [14444287374]
[ro.boottime.tz_service]: [11764145068]
[ro.boottime.tzts_service]: [11796108876]
[ro.boottime.ueventd]: [8289439071]
[ro.boottime.usbd]: [14734157066]
[ro.boottime.vaultkeeper]: [12265909606]
[ro.boottime.vaultkeeper_hal]: [12269328914]
[ro.boottime.vendor.audio-hal]: [13946017490]
[ro.boottime.vendor.bluetooth-1-0]: [13950598490]
[ro.boottime.vendor.cas-hal-1-2]: [13955020913]
[ro.boottime.vendor.configstore-hal]: [13959505567]
[ro.boottime.vendor.drm-clearkey-hal-1-3]: [13964087528]
[ro.boottime.vendor.drm-widevine-hal-1-3]: [13968351067]
[ro.boottime.vendor.face_hal]: [14774466258]
[ro.boottime.vendor.fps_hal]: [14780478181]
[ro.boottime.vendor.gatekeeper-1-0]: [13973933836]
[ro.boottime.vendor.gralloc-2-0]: [13978745182]
[ro.boottime.vendor.hwcomposer-2-2]: [13984484374]
[ro.boottime.vendor.keymaster-4-0]: [12055893376]
[ro.boottime.vendor.media.omx]: [14610806258]
[ro.boottime.vendor.memtrack-hal-1-0]: [13995538297]
[ro.boottime.vendor.samsung.hardware.hyper-default]: [14024679067]
[ro.boottime.vendor.samsung.hardware.light-default]: [14031006682]
[ro.boottime.vendor.samsung.power-default]: [14037669528]
[ro.boottime.vendor.sensors-hal-2-0-multihal]: [14000432720]
[ro.boottime.vendor.usb-hal-1-1]: [14009451567]
[ro.boottime.vendor.wifi_hal_ext]: [14109736336]
[ro.boottime.vendor.wifi_hal_legacy]: [14019783490]
[ro.boottime.vndservicemanager]: [10439621184]
[ro.boottime.vold]: [10596724838]
[ro.boottime.wificond]: [14605376874]
[ro.boottime.wlbtd]: [14675043528]
[ro.boottime.wpa_supplicant]: [27271643557]
[ro.boottime.zygote]: [13801121221]
[ro.boottime.zygote_secondary]: [13806564721]
[ro.bt.bdaddr_path]: [/mnt/vendor/efs/bluetooth/bt_addr]
[ro.build.2ndbrand]: [true]
[ro.build.PDA]: [A217NKSU5CUD3]
[ro.build.changelist]: [21209953]
[ro.build.characteristics]: [phone]
[ro.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.build.date.utc]: [1617760574]
[ro.build.description]: [a21sks-user 11 RP1A.200720.012 A217NKSU5CUD3 release-keys]
[ro.build.display.id]: [RP1A.200720.012.A217NKSU5CUD3]
[ro.build.display_build_number]: [true]
[ro.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.build.flavor]: [a21sks-user]
[ro.build.host]: [21DJ6814]
[ro.build.id]: [RP1A.200720.012]
[ro.build.keys]: [test-keys]
[ro.build.official.release]: [true]
[ro.build.product]: [a21s]
[ro.build.selinux]: [1]
[ro.build.selinux.enforce]: [1]
[ro.build.sw_version]: [A217NKSU5CUD3]
[ro.build.system_root_image]: [false]
[ro.build.tags]: [release-keys]
[ro.build.tf_modelnumber]: [SM-A217N]
[ro.build.type]: [user]
[ro.build.user]: [dpi]
[ro.build.version.all_codenames]: [REL]
[ro.build.version.base_os]: []
[ro.build.version.codename]: [REL]
[ro.build.version.incremental]: [A217NKSU5CUD3]
[ro.build.version.min_supported_target_sdk]: [23]
[ro.build.version.preview_sdk]: [0]
[ro.build.version.preview_sdk_fingerprint]: [REL]
[ro.build.version.release]: [11]
[ro.build.version.release_or_codename]: [11]
[ro.build.version.sdk]: [30]
[ro.build.version.security_index]: [2]
[ro.build.version.security_patch]: [2021-03-01]
[ro.build.version.sem]: [3001]
[ro.build.version.sep]: [120100]
[ro.camera.notify_nfc]: [1]
[ro.carrier]: [unknown]
[ro.channel.officehubrow]: [SAMSUNG_PRELOAD]
[ro.cmc.device_type]: [pd]
[ro.cmc.version]: [2.0]
[ro.cold_boot_done]: [true]
[ro.com.google.clientidbase]: [android-samsung-ss]
[ro.com.google.clientidbase.am]: [android-lgu-kr]
[ro.com.google.gmsversion]: [11_202101]
[ro.config.alarm_alert]: [Homecoming.ogg]
[ro.config.dmverity]: [G]
[ro.config.iccc_version]: [3.0]
[ro.config.knox]: [v30]
[ro.config.knox.ucm]: [1]
[ro.config.media_sound]: [Media_preview_Touch_the_light.ogg]
[ro.config.notification_sound]: [Spaceline.ogg]
[ro.config.notification_sound_2]: [Signal.ogg]
[ro.config.pageboost.vramdisk.bootfile.enabled]: [true]
[ro.config.pageboost.vramdisk.minimize]: [true]
[ro.config.ringtone]: [Over_the_Horizon.ogg]
[ro.config.ringtone_2]: [Atomic_Bell.ogg]
[ro.config.rm_preload_enabled]: [1]
[ro.config.systemaudiodebug]: [abox&codecdsp]
[ro.config.tima]: [1]
[ro.config.timaversion]: [3.0]
[ro.config.timaversion_info]: [Knox3.4_S.R.R]
[ro.config.vc_call_vol_steps]: [8]
[ro.control_privapp_permissions]: [enforce]
[ro.crypto.state]: [unsupported]
[ro.crypto.volume.filenames_mode]: [aes-256-cts]
[ro.csc.country_code]: [KOREA]
[ro.csc.countryiso_code]: [KR]
[ro.csc.facebook.partnerid]: [samsung:dec1cc9c-1497-4aab-b953-cee702c2a481]
[ro.csc.omcnw_code]: [SKC]
[ro.csc.sales_code]: [SKC]
[ro.dalvik.vm.native.bridge]: [0]
[ro.debuggable]: [0]
[ro.em.version]: [20]
[ro.frp.pst]: [/dev/block/persistent]
[ro.gsid.image_running]: [0]
[ro.hardware]: [exynos850]
[ro.hardware.chipname]: [exynos850]
[ro.hardware.keystore]: [mdfpp]
[ro.hwui.use_vulkan]: []
[ro.iorapd.enable]: [false]
[ro.kernel.qemu]: [0]
[ro.kernel.qemu.gles]: [2]
[ro.lmk.use_minfree_levels]: [true]
[ro.logd.auditd]: [false]
[ro.logd.size.stats]: [64K]
[ro.multisim.simslotcount]: [1]
[ro.odm.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.odm.build.date.utc]: [1617760574]
[ro.odm.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.odm.build.id]: [RP1A.200720.012]
[ro.odm.build.tags]: [release-keys]
[ro.odm.build.type]: [user]
[ro.odm.build.version.incremental]: [A217NKSU5CUD3]
[ro.odm.build.version.release]: [11]
[ro.odm.build.version.release_or_codename]: [11]
[ro.odm.build.version.sdk]: [30]
[ro.odm.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.odm.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.odm.product.cpu.abilist64]: [arm64-v8a]
[ro.oem.key1]: [SKC]
[ro.oem_unlock_supported]: [1]
[ro.omc.build.id]: [39385906]
[ro.omc.build.version]: [A217NOKR5CUD3]
[ro.omc.changetype]: [DATA_RESET_ON,FALSE]
[ro.omc.disabler]: [TRUE]
[ro.omc.img_mount]: [0]
[ro.omc.region]: [KOR]
[ro.opa.eligible_device]: [true]
[ro.opengles.version]: [196610]
[ro.organization_owned]: [false]
[ro.persistent_properties.ready]: [true]
[ro.postinstall.fstab.prefix]: [/system]
[ro.product.board]: [exynos850]
[ro.product.brand]: [samsung]
[ro.product.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.product.build.date.utc]: [1617760574]
[ro.product.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.product.build.id]: [RP1A.200720.012]
[ro.product.build.tags]: [release-keys]
[ro.product.build.type]: [user]
[ro.product.build.version.incremental]: [A217NKSU5CUD3]
[ro.product.build.version.release]: [11]
[ro.product.build.version.release_or_codename]: [11]
[ro.product.build.version.sdk]: [30]
[ro.product.cpu.abi]: [arm64-v8a]
[ro.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.product.cpu.abilist64]: [arm64-v8a]
[ro.product.device]: [a21s]
[ro.product.first_api_level]: [29]
[ro.product.locale]: [ko-KR]
[ro.product.manufacturer]: [samsung]
[ro.product.model]: [SM-A217N]
[ro.product.name]: [a21sks]
[ro.product.odm.brand]: [samsung]
[ro.product.odm.device]: [a21s]
[ro.product.odm.manufacturer]: [samsung]
[ro.product.odm.model]: [SM-A217N]
[ro.product.odm.name]: [a21sks]
[ro.product.product.brand]: [samsung]
[ro.product.product.device]: [a21s]
[ro.product.product.manufacturer]: [samsung]
[ro.product.product.model]: [SM-A217N]
[ro.product.product.name]: [a21sks]
[ro.product.system.brand]: [samsung]
[ro.product.system.device]: [a21s]
[ro.product.system.manufacturer]: [samsung]
[ro.product.system.model]: [SM-A217N]
[ro.product.system.name]: [a21sks]
[ro.product.system_ext.brand]: [samsung]
[ro.product.system_ext.device]: [a21s]
[ro.product.system_ext.manufacturer]: [samsung]
[ro.product.system_ext.model]: [SM-A217N]
[ro.product.system_ext.name]: [a21sks]
[ro.product.vendor.brand]: [samsung]
[ro.product.vendor.device]: [a21s]
[ro.product.vendor.manufacturer]: [samsung]
[ro.product.vendor.model]: [SM-A217N]
[ro.product.vendor.name]: [a21sks]
[ro.product_ship]: [true]
[ro.property_service.version]: [2]
[ro.radio.noril]: [no]
[ro.revision]: [6]
[ro.ril.def_network_after_check_tdscdma]: [9]
[ro.ril.support_cdma]: [false]
[ro.ril.svdo]: [false]
[ro.ril.svlte1x]: [false]
[ro.runtime.firstboot]: [1621089963878]
[ro.secure]: [1]
[ro.security.cass.feature]: [1]
[ro.security.keystore.keytype]: [sakv2,gak]
[ro.security.reactive.version]: [2.0.12]
[ro.security.vpnpp.release]: [1.0]
[ro.security.vpnpp.ver]: [2.2]
[ro.serialno]: [RF9N8010NYL]
[ro.setupwizard.mode]: [OPTIONAL]
[ro.setupwizard.rotation_locked]: [false]
[ro.setupwizard.wifi_on_exit]: [false]
[ro.sf.lcd_density]: [280]
[ro.simbased.changetype]: [NO_DFLT_CSC,OMC]
[ro.slmk.2nd.dha_cached_max]: [14]
[ro.slmk.2nd.dha_empty_max]: [24]
[ro.slmk.2nd.freelimit_val]: [12]
[ro.slmk.2nd.swap_free_low_percentage]: [10]
[ro.slmk.3rd.dha_cached_max]: [24]
[ro.slmk.3rd.dha_cached_min]: [6]
[ro.slmk.3rd.empty_max]: [30]
[ro.slmk.3rd.over_thMB]: [4096]
[ro.slmk.add_bonusEFK]: [1]
[ro.slmk.dha_2ndprop_thMB]: [3072]
[ro.slmk.dha_cached_max]: [10]
[ro.slmk.dha_cached_min]: [3]
[ro.slmk.dha_empty_init]: [20]
[ro.slmk.dha_empty_max]: [20]
[ro.slmk.dha_pwhl_key]: [520]
[ro.slmk.freelimit_val]: [13]
[ro.slmk.plg_key]: [1]
[ro.slmk.plg_memory_th]: [2048]
[ro.slmk.psi_critical]: [150]
[ro.slmk.psi_medium]: [100]
[ro.slmk.swap_free_low_percentage]: [20]
[ro.slmk.use_lowmem_keep_except]: [true]
[ro.sts.property]: [FALSE]
[ro.surface_flinger.max_frame_buffer_acquired_buffers]: [3]
[ro.surface_flinger.running_without_sync_framework]: [false]
[ro.surface_flinger.vsync_event_phase_offset_ns]: [0]
[ro.surface_flinger.vsync_sf_event_phase_offset_ns]: [0]
[ro.sys.kernelmemory.nandswap.daily_quota]: [131072]
[ro.sys.kernelmemory.nandswap.daily_quota_limit]: [393216]
[ro.system.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.system.build.date.utc]: [1617760574]
[ro.system.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.system.build.id]: [RP1A.200720.012]
[ro.system.build.tags]: [release-keys]
[ro.system.build.type]: [user]
[ro.system.build.version.incremental]: [A217NKSU5CUD3]
[ro.system.build.version.release]: [11]
[ro.system.build.version.release_or_codename]: [11]
[ro.system.build.version.sdk]: [30]
[ro.system.build.version.sehi]: [3003]
[ro.system_ext.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.system_ext.build.date.utc]: [1617760574]
[ro.system_ext.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.system_ext.build.id]: [RP1A.200720.012]
[ro.system_ext.build.tags]: [release-keys]
[ro.system_ext.build.type]: [user]
[ro.system_ext.build.version.incremental]: [A217NKSU5CUD3]
[ro.system_ext.build.version.release]: [11]
[ro.system_ext.build.version.release_or_codename]: [11]
[ro.system_ext.build.version.sdk]: [30]
[ro.telephony.default_cdma_sub]: [0]
[ro.telephony.default_network]: [9]
[ro.telephony.iwlan_operation_mode]: [legacy]
[ro.treble.enabled]: [true]
[ro.user_trial]: [FALSE]
[ro.vendor.boot.carrierid.param.offset]: [7340596]
[ro.vendor.boot.debug_level]: [0x4f4c]
[ro.vendor.boot.force_upload]: [0x0]
[ro.vendor.boot.im.param.offset]: [7342864]
[ro.vendor.boot.me.param.offset]: [7342944]
[ro.vendor.boot.sales.param.offset]: [7340568]
[ro.vendor.boot.sn.param.offset]: [7343024]
[ro.vendor.boot.warranty_bit]: [1]
[ro.vendor.build.characteristics]: [phone]
[ro.vendor.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.vendor.build.date.utc]: [1617760574]
[ro.vendor.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.vendor.build.id]: [RP1A.200720.012]
[ro.vendor.build.security_patch]: []
[ro.vendor.build.tags]: [release-keys]
[ro.vendor.build.type]: [user]
[ro.vendor.build.version.incremental]: [A217NKSU5CUD3]
[ro.vendor.build.version.release]: [11]
[ro.vendor.build.version.release_or_codename]: [11]
[ro.vendor.build.version.sdk]: [30]
[ro.vendor.config.release_version]: [20190306]
[ro.vendor.cscsupported]: [1]
[ro.vendor.epdg.support]: [true]
[ro.vendor.multisim.simslotcount]: [1]
[ro.vendor.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.vendor.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.vendor.product.cpu.abilist64]: [arm64-v8a]
[ro.vendor.product_ship]: [true]
[ro.vendor.radio.default_network]: [9]
[ro.vendor.radio.default_network_by]: [telephony.prop_9]
[ro.vendor.redirect_socket_calls]: [true]
[ro.vendor.sec.radio.def_network]: [9]
[ro.vendor.security.vaultkeeper.id]: [Qq61v94394jM79gM]
[ro.vndk.version]: [30]
[ro.wifi.channels]: []
[ro.zygote]: [zygote64_32]
[secmm.codecsolution.ready]: [1]
[security.ADP.policy_version]: [20191001]
[security.ADP.version]: [3.0]
[security.ASKS.policy_version]: [20200806]
[security.ASKS.rufs_enable]: [true]
[security.ASKS.time_value]: [20210516]
[security.ASKS.version]: [4.0]
[security.perf_harden]: [1]
[security.securehw.available]: [false]
[security.securenvm.available]: [false]
[security.ucm_version]: [1.11]
[selinux.policy_version]: [SEPF_SM-A217N_11_0009]
[selinux.restorecon_recursive]: [/data/misc_ce/0]
[service.bootanim.exit]: [1]
[service.camera.running]: [0]
[service.media.powersnd]: [1]
[service.sf.present_timestamp]: [1]
[setupwizard.feature.enable_stencil_partner_customization]: [1]
[setupwizard.theme]: [glif_v3_light]
[storage.support.sdcard]: [1]
[storage.support.usb]: [1]
[sys.boot.errp]: [1]
[sys.boot.reason]: [reboot,shit]
[sys.boot.reason.last]: [reboot,shit]
[sys.boot_completed]: [1]
[sys.bootstat.first_boot_completed]: [1]
[sys.config.activelaunch_enable]: [true]
[sys.config.hardcoder.enable]: [false]
[sys.config.ibs.enable]: [false]
[sys.config.mars_version]: [6.00]
[sys.default.data.phoneid]: [0]
[sys.dualapp.profile_id]: []
[sys.enterprise.billing.dualsim]: [true]
[sys.enterprise.billing.version]: [1.3.0]
[sys.factory.secphone_ready]: [true]
[sys.ims.nsd.first_trigger]: [false]
[sys.is_members]: [exist]
[sys.is_vold_shutdown]: [0]
[sys.isolated_storage_snapshot]: [true]
[sys.lmk.minfree_levels]: [18432:0,23040:100,27648:200,32256:250,55296:900,80640:950]
[sys.lmk.reportkills]: [1]
[sys.members.installed]: [true]
[sys.oem_unlock_allowed]: [1]
[sys.pdp.action]: [cekeyunlock]
[sys.perf.aaal]: [true]
[sys.perf.aal]: [true]
[sys.perf.aid]: [true]
[sys.perf.bc]: [true]
[sys.perf.cg]: [true]
[sys.perf.fbooster]: [true]
[sys.perf.gvn]: [true]
[sys.perf.heavy]: [false]
[sys.perf.olaf]: [true]
[sys.perf.schd]: [true]
[sys.perf.status]: [false]
[sys.perf.tbooster]: [true]
[sys.perf.zygote]: [true]
[sys.rdxd.create]: [1]
[sys.rdxd.erase_all]: [1]
[sys.rdxd.total_size]: [4]
[sys.rescue_boot_count]: [1]
[sys.reset_reason]: []
[sys.retaildemo.enabled]: [0]
[sys.siop.level]: [-3]
[sys.smf.mnoname0]: [SKT_KR|LOADED]
[sys.sysctl.extra_free_kbytes]: [40892]
[sys.sysctl.tcp_def_init_rwnd]: [60]
[sys.sysctl.watermark_boost_factor]: [0]
[sys.system_server.start_count]: [1]
[sys.system_server.start_elapsed]: [17181]
[sys.system_server.start_uptime]: [17181]
[sys.usb.config]: [mtp,adb]
[sys.usb.configfs]: [2]
[sys.usb.configured]: [configured]
[sys.usb.controller]: [13600000.dwc3]
[sys.usb.ffs.ready]: [1]
[sys.usb.state]: [mtp,adb]
[sys.use_memfd]: [false]
[sys.user.0.ce_available]: [true]
[vendor.TEEGRIS.loaded]: [true]
[vendor.bluetooth_fw_ver]: [UTC|nacho_s612|S612_E3830_R4-1_Y21W10A]
[vendor.cbd.boot_done]: [1]
[vendor.cbd.cp_reset]: [1]
[vendor.cbd.first_xmit_done]: [1]
[vendor.cbd.rfs_check_done]: [1]
[vendor.gsm.sim.state]: [LOADED]
[vendor.gsm.version.baseband]: [A217NKOU5CUD1]
[vendor.ril.sw_ver]: [A217NKOU5CUD1]
[vendor.sec.rild.libpath]: [/vendor/lib64/libsec-ril.so]
[vendor.secureos.kernel.version]: [Samsung Secure OS Release Version 4.1.0.0 built on: 2020-11-04 11:09:29, binary version: 10]
[vendor.secureos.nwd_libs.version]: [4.1.0 Nov  4 2020 10:51:09 ]
[vendor.secureos.tzdaemon.version]: [4.1.0 compiled on: Nov  4 2020 10:51:09]
[vendor.security.vaultkeeper.native]: [1]
[vendor.tzdaemon]: [Ready]
[vendor.tzts_daemon]: [Ready]
[vendor.wifi.dualconcurrent.interface]: [swlan0]
[vendor.wifiap.defaultmaxclient]: [10]
[vendor.wifiap.wifisharinglite]: [true]
[vendor.wlan.driver.version]: [drv_ver: 11.13.0.0.0]
[vendor.wlan.firmware.version]: [O|6714|2021-03-09 20:12:01 UTC|nacho_s612|S612_E3830_R4-1_Y21W10A]
[vendor.wlbtd.board_id]: [0]
[vendor.wlbtd.tar_files_per_trigger]: [2]
[vendor.wlbtd.version]: [3.8]
[vold.has_adoptable]: [1]
[vold.has_quota]: [1]
[vold.has_reserved]: [1]
[vold.post_fs_data_done]: [1]
[wifi.active.interface]: []
[wifi.interface]: [wlan0]
[wlan.driver.status]: [unloaded]
[wlan.p2p.chkintent]: [0]
[wlan.p2p.temp]: [-3]
[wlan.p2p.wfdsta]: [disconnected]

This is the dmesg I got from rooted A217N stock UD3(R)

Reply:
a21s:/ # getprop
[aaudio.hw_burst_min_usec]: [2000]
[aaudio.mmap_exclusive_policy]: [2]
[aaudio.mmap_policy]: [2]
[apexd.status]: [ready]
[boot.sfbootcomplete]: [0]
[bpf.progs_loaded]: [1]
[build.version.extensions.r]: [0]
[cache_key.bluetooth.get_bond_state]: [9198186490379997408]
[cache_key.bluetooth.get_profile_connection_state]: [9198186490379997411]
[cache_key.bluetooth.get_state]: [9198186490379997413]
[cache_key.bluetooth.is_offloaded_filtering_supported]: [9198186490379997412]
[cache_key.display_info]: [-2274032986854524931]
[cache_key.has_system_feature]: [-2274032986854525094]
[cache_key.is_compat_change_enabled]: [-2274032986854524940]
[cache_key.is_interactive]: [-2274032986854524933]
[cache_key.is_power_save_mode]: [-2274032986854525085]
[cache_key.is_user_unlocked]: [-2274032986854525078]
[cache_key.location_enabled]: [-2274032986854525084]
[cache_key.package_info]: [-2274032986854524930]
[cache_key.telephony.get_active_data_sub_id]: [4052843020917718026]
[cache_key.telephony.get_default_data_sub_id]: [4052843020917717920]
[cache_key.telephony.get_default_sms_sub_id]: [4052843020917717924]
[cache_key.telephony.get_default_sub_id]: [4052843020917717925]
[cache_key.telephony.get_slot_index]: [4052843020917717926]
[com.samsung.speg.disable]: [true]
[dalvik.vm.appimageformat]: [lz4]
[dalvik.vm.dex2oat-Xms]: [64m]
[dalvik.vm.dex2oat-Xmx]: [512m]
[dalvik.vm.dex2oat-max-image-block-size]: [524288]
[dalvik.vm.dex2oat-minidebuginfo]: [true]
[dalvik.vm.dex2oat-resolve-startup-strings]: [true]
[dalvik.vm.dex2oat-updatable-bcp-packages-file]: [/system/etc/updatable-bcp-packages.txt]
[dalvik.vm.dexopt.secondary]: [true]
[dalvik.vm.heapgrowthlimit]: [192m]
[dalvik.vm.heapmaxfree]: [8m]
[dalvik.vm.heapminfree]: [512k]
[dalvik.vm.heapsize]: [512m]
[dalvik.vm.heapstartsize]: [8m]
[dalvik.vm.heaptargetutilization]: [0.75]
[dalvik.vm.image-dex2oat-Xms]: [64m]
[dalvik.vm.image-dex2oat-Xmx]: [64m]
[dalvik.vm.isa.arm.features]: [default]
[dalvik.vm.isa.arm.variant]: [cortex-a15]
[dalvik.vm.isa.arm64.features]: [default]
[dalvik.vm.isa.arm64.variant]: [generic]
[dalvik.vm.lockprof.threshold]: [500]
[dalvik.vm.minidebuginfo]: [true]
[dalvik.vm.usejit]: [true]
[dalvik.vm.usejitprofiles]: [true]
[debug.atrace.tags.enableflags]: [0]
[debug.force_rtl]: [false]
[debug.hwc.winupdate]: [1]
[debug.sf.disable_backpressure]: [1]
[debug.sf.high_fps_late_app_phase_offset_ns]: [0]
[debug.sf.high_fps_late_sf_phase_offset_ns]: [0]
[debug.sf.latch_unsignaled]: [1]
[debug.slsi_platform]: [1]
[debug.tracing.screen_brightness]: [0.0]
[debug.tracing.screen_state]: [1]
[dev.bootcomplete]: [1]
[dev.kies.drivedisplay.trust]: [1]
[dev.mnt.blk.odm]: [dm-3]
[dev.mnt.blk.product]: [dm-2]
[dev.mnt.blk.root]: [dm-0]
[dev.mnt.blk.vendor]: [dm-1]
[dev.sdhms.ltb]: [0,510:1,410:PM:]
[dev.sdhms.ss]: [0,770:1,550:]
[dev.silentlog.on]: []
[dev.ssrm.app.install.standby]: [-1]
[dev.ssrm.appsync3p]: [true]
[dev.ssrm.gamelevel]: [-5,0,-2,4]
[dev.ssrm.init]: [1]
[dev.ssrm.live_thumbnail]: [1]
[dev.ssrm.turbo]: [false]
[dev.ssrm.version]: [4.0]
[dev.usbsetting.embedded]: [on]
[graphics.display.kernel_idle_timer.enabled]: [false]
[gsid.image_installed]: [0]
[gsm.STK_SETUP_MENU]: [T USIM 서비스]
[gsm.current.phone-type]: [1]
[gsm.facilitylock.state]: [false]
[gsm.network.type]: [LTE]
[gsm.nitz.time]: [1621124413000]
[gsm.nitz.time-elapsedtime]: [34497901]
[gsm.operator.alpha]: [SKTelecom]
[gsm.operator.iso-country]: [kr]
[gsm.operator.isroaming]: [false]
[gsm.operator.numeric]: [45005]
[gsm.sim.acc]: [45005]
[gsm.sim.operator.alpha]: [SK Telecom]
[gsm.sim.operator.iso-country]: [kr]
[gsm.sim.operator.numeric]: [45005]
[gsm.sim.sprintsim0]: [0]
[gsm.sim.state]: [LOADED]
[gsm.version.baseband]: [A217NKOU5CUD1]
[gsm.version.ril-impl]: [Samsung RIL v4.0]
[hwservicemanager.ready]: [true]
[init.svc.BCS-daemon]: [running]
[init.svc.DIAG-daemon]: [running]
[init.svc.DR-daemon]: [running]
[init.svc.ExynosHWCConfigs]: [running]
[init.svc.ExynosHWCServiceTW]: [running]
[init.svc.HTduikIjCS]: [stopped]
[init.svc.SMD-daemon]: [running]
[init.svc.abox]: [running]
[init.svc.abox_log]: [running]
[init.svc.adbd]: [running]
[init.svc.apexd]: [stopped]
[init.svc.apexd-bootstrap]: [stopped]
[init.svc.apexd-snapshotde]: [stopped]
[init.svc.argos-daemon]: [running]
[init.svc.at_distributor]: [stopped]
[init.svc.atnserv]: [stopped]
[init.svc.audioserver]: [running]
[init.svc.auditd]: [stopped]
[init.svc.bootanim]: [stopped]
[init.svc.bootchecker]: [stopped]
[init.svc.bootchecker-bootc]: [stopped]
[init.svc.boringssl_self_test32]: [stopped]
[init.svc.boringssl_self_test32_vendor]: [stopped]
[init.svc.boringssl_self_test64]: [stopped]
[init.svc.boringssl_self_test64_vendor]: [stopped]
[init.svc.boringssl_self_test_apex32]: [stopped]
[init.svc.boringssl_self_test_apex64]: [stopped]
[init.svc.bpfloader]: [stopped]
[init.svc.cameraserver]: [running]
[init.svc.cass]: [running]
[init.svc.cpboot-daemon]: [running]
[init.svc.credstore]: [running]
[init.svc.dNMYGJYXgqm4IK]: [stopped]
[init.svc.derive_sdk]: [stopped]
[init.svc.drk_hal]: [running]
[init.svc.drm]: [running]
[init.svc.dsmsd]: [running]
[init.svc.emservice]: [running]
[init.svc.engmode_hal]: [running]
[init.svc.flash_recovery_sec]: [stopped]
[init.svc.gatekeeperd]: [running]
[init.svc.gpsd]: [running]
[init.svc.gpu]: [running]
[init.svc.health-hal-2-1-samsung]: [running]
[init.svc.heapprofd]: [stopped]
[init.svc.hidl_memory]: [running]
[init.svc.hqmserv]: [running]
[init.svc.hwservicemanager]: [running]
[init.svc.idmap2d]: [stopped]
[init.svc.imsd]: [running]
[init.svc.incidentd]: [running]
[init.svc.installd]: [running]
[init.svc.insthk]: [stopped]
[init.svc.iod]: [running]
[init.svc.iorapd]: [stopped]
[init.svc.keystore]: [running]
[init.svc.lmkd]: [running]
[init.svc.logd]: [running]
[init.svc.logd-auditctl]: [stopped]
[init.svc.logd-reinit]: [stopped]
[init.svc.macloader]: [stopped]
[init.svc.media]: [running]
[init.svc.media.swcodec]: [running]
[init.svc.mediadrm]: [running]
[init.svc.mediaextractor]: [running]
[init.svc.mediametrics]: [running]
[init.svc.multiclient-daemon]: [running]
[init.svc.netd]: [running]
[init.svc.network_config]: [stopped]
[init.svc.nfc_hal_service]: [running]
[init.svc.pOmHHmDQ7hneg]: [stopped]
[init.svc.perfsdkserver]: [running]
[init.svc.powersnd]: [stopped]
[init.svc.prepare_param]: [stopped]
[init.svc.rdxd]: [stopped]
[init.svc.resetreason]: [stopped]
[init.svc.ril-daemon]: [running]
[init.svc.ril-daemon1]: [stopped]
[init.svc.scs]: [stopped]
[init.svc.sdp_cryptod]: [running]
[init.svc.sec-camera-provider-4-0]: [running]
[init.svc.sec-miscpower-2-0]: [running]
[init.svc.sec-sysinput-1-1]: [running]
[init.svc.sec-thermal-1-0]: [running]
[init.svc.sec-vibrator-2-2]: [running]
[init.svc.sec_gnss_service]: [running]
[init.svc.servicemanager]: [running]
[init.svc.sim_config]: [stopped]
[init.svc.snap_utility_64]: [stopped]
[init.svc.snapserv]: [running]
[init.svc.ss_conn_daemon2_service]: [stopped]
[init.svc.statsd]: [running]
[init.svc.storaged]: [running]
[init.svc.storkd]: [running]
[init.svc.surfaceflinger]: [running]
[init.svc.system_suspend]: [running]
[init.svc.timahidlserv]: [stopped]
[init.svc.tombstoned]: [running]
[init.svc.traced]: [running]
[init.svc.traced_perf]: [stopped]
[init.svc.traced_probes]: [running]
[init.svc.tz_service]: [running]
[init.svc.tzts_service]: [running]
[init.svc.ueventd]: [running]
[init.svc.usbd]: [stopped]
[init.svc.vaultkeeper]: [running]
[init.svc.vaultkeeper_hal]: [running]
[init.svc.vendor.audio-hal]: [running]
[init.svc.vendor.bluetooth-1-0]: [running]
[init.svc.vendor.cas-hal-1-2]: [running]
[init.svc.vendor.configstore-hal]: [running]
[init.svc.vendor.drm-clearkey-hal-1-3]: [running]
[init.svc.vendor.drm-widevine-hal-1-3]: [running]
[init.svc.vendor.face_hal]: [running]
[init.svc.vendor.fps_hal]: [running]
[init.svc.vendor.gatekeeper-1-0]: [running]
[init.svc.vendor.gralloc-2-0]: [running]
[init.svc.vendor.hwcomposer-2-2]: [running]
[init.svc.vendor.keymaster-4-0]: [running]
[init.svc.vendor.media.omx]: [running]
[init.svc.vendor.memtrack-hal-1-0]: [running]
[init.svc.vendor.samsung.hardware.hyper-default]: [running]
[init.svc.vendor.samsung.hardware.light-default]: [running]
[init.svc.vendor.samsung.power-default]: [running]
[init.svc.vendor.sensors-hal-2-0-multihal]: [running]
[init.svc.vendor.usb-hal-1-1]: [running]
[init.svc.vendor.wifi_hal_ext]: [running]
[init.svc.vendor.wifi_hal_legacy]: [running]
[init.svc.vndservicemanager]: [running]
[init.svc.vold]: [running]
[init.svc.wificond]: [running]
[init.svc.wlbtd]: [running]
[init.svc.wpa_supplicant]: [stopped]
[init.svc.zygote]: [running]
[init.svc.zygote_secondary]: [running]
[init.svc_debug_pid.BCS-daemon]: [4918]
[init.svc_debug_pid.DIAG-daemon]: [4914]
[init.svc_debug_pid.DR-daemon]: [4915]
[init.svc_debug_pid.ExynosHWCConfigs]: [4852]
[init.svc_debug_pid.ExynosHWCServiceTW]: [4848]
[init.svc_debug_pid.HTduikIjCS]: []
[init.svc_debug_pid.SMD-daemon]: [4913]
[init.svc_debug_pid.abox]: [4975]
[init.svc_debug_pid.abox_log]: [4977]
[init.svc_debug_pid.adbd]: [4550]
[init.svc_debug_pid.apexd]: []
[init.svc_debug_pid.apexd-bootstrap]: []
[init.svc_debug_pid.apexd-snapshotde]: []
[init.svc_debug_pid.argos-daemon]: [4937]
[init.svc_debug_pid.at_distributor]: []
[init.svc_debug_pid.atnserv]: []
[init.svc_debug_pid.audioserver]: [4855]
[init.svc_debug_pid.auditd]: []
[init.svc_debug_pid.bootanim]: []
[init.svc_debug_pid.bootchecker]: []
[init.svc_debug_pid.bootchecker-bootc]: []
[init.svc_debug_pid.boringssl_self_test32]: []
[init.svc_debug_pid.boringssl_self_test32_vendor]: []
[init.svc_debug_pid.boringssl_self_test64]: []
[init.svc_debug_pid.boringssl_self_test64_vendor]: []
[init.svc_debug_pid.boringssl_self_test_apex32]: []
[init.svc_debug_pid.boringssl_self_test_apex64]: []
[init.svc_debug_pid.bpfloader]: []
[init.svc_debug_pid.cameraserver]: [4908]
[init.svc_debug_pid.cass]: [4752]
[init.svc_debug_pid.cpboot-daemon]: [4938]
[init.svc_debug_pid.credstore]: [4865]
[init.svc_debug_pid.dNMYGJYXgqm4IK]: []
[init.svc_debug_pid.derive_sdk]: []
[init.svc_debug_pid.drk_hal]: [4832]
[init.svc_debug_pid.drm]: [4909]
[init.svc_debug_pid.dsmsd]: [4882]
[init.svc_debug_pid.emservice]: [4753]
[init.svc_debug_pid.engmode_hal]: [4513]
[init.svc_debug_pid.flash_recovery_sec]: []
[init.svc_debug_pid.gatekeeperd]: [4969]
[init.svc_debug_pid.gpsd]: [4944]
[init.svc_debug_pid.gpu]: [4872]
[init.svc_debug_pid.health-hal-2-1-samsung]: [4807]
[init.svc_debug_pid.heapprofd]: []
[init.svc_debug_pid.hidl_memory]: [4790]
[init.svc_debug_pid.hqmserv]: [5294]
[init.svc_debug_pid.hwservicemanager]: [4342]
[init.svc_debug_pid.idmap2d]: []
[init.svc_debug_pid.imsd]: [4911]
[init.svc_debug_pid.incidentd]: [4912]
[init.svc_debug_pid.installd]: [4919]
[init.svc_debug_pid.insthk]: []
[init.svc_debug_pid.iod]: [4482]
[init.svc_debug_pid.iorapd]: []
[init.svc_debug_pid.keystore]: [4920]
[init.svc_debug_pid.lmkd]: [4340]
[init.svc_debug_pid.logd]: [4339]
[init.svc_debug_pid.logd-auditctl]: []
[init.svc_debug_pid.logd-reinit]: []
[init.svc_debug_pid.macloader]: []
[init.svc_debug_pid.media]: [4931]
[init.svc_debug_pid.media.swcodec]: [4955]
[init.svc_debug_pid.mediaextractor]: [4925]
[init.svc_debug_pid.mediametrics]: [4929]
[init.svc_debug_pid.multiclient-daemon]: [5145]
[init.svc_debug_pid.netd]: [4774]
[init.svc_debug_pid.network_config]: []
[init.svc_debug_pid.nfc_hal_service]: [5954]
[init.svc_debug_pid.pOmHHmDQ7hneg]: []
[init.svc_debug_pid.perfsdkserver]: [6649]
[init.svc_debug_pid.powersnd]: []
[init.svc_debug_pid.prepare_param]: []
[init.svc_debug_pid.rdxd]: []
[init.svc_debug_pid.resetreason]: []
[init.svc_debug_pid.ril-daemon]: [4947]
[init.svc_debug_pid.ril-daemon1]: []
[init.svc_debug_pid.scs]: []
[init.svc_debug_pid.sdp_cryptod]: [4907]
[init.svc_debug_pid.sec-camera-provider-4-0]: [4827]
[init.svc_debug_pid.sec-miscpower-2-0]: [4831]
[init.svc_debug_pid.sec-sysinput-1-1]: [4837]
[init.svc_debug_pid.sec-thermal-1-0]: [4840]
[init.svc_debug_pid.sec-vibrator-2-2]: [4844]
[init.svc_debug_pid.sec_gnss_service]: [4948]
[init.svc_debug_pid.servicemanager]: [4341]
[init.svc_debug_pid.sim_config]: []
[init.svc_debug_pid.snap_utility_64]: []
[init.svc_debug_pid.snapserv]: [4836]
[init.svc_debug_pid.ss_conn_daemon2_service]: []
[init.svc_debug_pid.statsd]: [4772]
[init.svc_debug_pid.storaged]: [4932]
[init.svc_debug_pid.storkd]: [4525]
[init.svc_debug_pid.surfaceflinger]: [4876]
[init.svc_debug_pid.system_suspend]: [4511]
[init.svc_debug_pid.timahidlserv]: []
[init.svc_debug_pid.tombstoned]: [4751]
[init.svc_debug_pid.traced]: [4903]
[init.svc_debug_pid.traced_perf]: []
[init.svc_debug_pid.traced_probes]: [4902]
[init.svc_debug_pid.tz_service]: [4485]
[init.svc_debug_pid.tzts_service]: [4488]
[init.svc_debug_pid.ueventd]: [3137]
[init.svc_debug_pid.usbd]: []
[init.svc_debug_pid.vaultkeeper]: [4523]
[init.svc_debug_pid.vaultkeeper_hal]: [4524]
[init.svc_debug_pid.vendor.audio-hal]: [4791]
[init.svc_debug_pid.vendor.bluetooth-1-0]: [4792]
[init.svc_debug_pid.vendor.cas-hal-1-2]: [4793]
[init.svc_debug_pid.vendor.configstore-hal]: [4794]
[init.svc_debug_pid.vendor.drm-clearkey-hal-1-3]: [4795]
[init.svc_debug_pid.vendor.drm-widevine-hal-1-3]: [4796]
[init.svc_debug_pid.vendor.face_hal]: [4978]
[init.svc_debug_pid.vendor.fps_hal]: [4980]
[init.svc_debug_pid.vendor.gatekeeper-1-0]: [4797]
[init.svc_debug_pid.vendor.gralloc-2-0]: [4798]
[init.svc_debug_pid.vendor.hwcomposer-2-2]: [4804]
[init.svc_debug_pid.vendor.keymaster-4-0]: [4512]
[init.svc_debug_pid.vendor.media.omx]: [4935]
[init.svc_debug_pid.vendor.memtrack-hal-1-0]: [4808]
[init.svc_debug_pid.vendor.samsung.hardware.hyper-default]: [4823]
[init.svc_debug_pid.vendor.samsung.hardware.light-default]: [4824]
[init.svc_debug_pid.vendor.samsung.power-default]: [4825]
[init.svc_debug_pid.vendor.sensors-hal-2-0-multihal]: [4810]
[init.svc_debug_pid.vendor.usb-hal-1-1]: [4820]
[init.svc_debug_pid.vendor.wifi_hal_ext]: [4845]
[init.svc_debug_pid.vendor.wifi_hal_legacy]: [4822]
[init.svc_debug_pid.vndservicemanager]: [4344]
[init.svc_debug_pid.vold]: [4361]
[init.svc_debug_pid.wificond]: [4934]
[init.svc_debug_pid.wlbtd]: [4953]
[init.svc_debug_pid.wpa_supplicant]: []
[init.svc_debug_pid.zygote]: [4775]
[init.svc_debug_pid.zygote_secondary]: [4776]
[keyguard.no_require_sim]: [true]
[log.tag.APM_AudioPolicyManager]: [V]
[log.tag.snap_api::V3]: [WARN]
[log.tag.snap_api::snap_eden::V3]: [WARN]
[log.tag.snap_api::snap_ofi::V1]: [WARN]
[log.tag.snap_api::snpe]: [WARN]
[log.tag.snap_compute::V2]: [WARN]
[log.tag.snap_compute::V3]: [WARN]
[log.tag.snap_hidl_v3]: [WARN]
[log.tag.snap_service@1.2]: [WARN]
[log.tag.snaplite_lib]: [WARN]
[log.tag.stats_log]: [I]
[mdc.omc_root]: [/prism/etc/carriers]
[mdc.scs.running_state]: [4]
[mdc.scs.tss_checked]: [1]
[mdc.singlesku]: [false]
[mdc.sys.enable_smff]: [false]
[mdc.sys.sec_cid]: [30242]
[mdc.sys.sec_pcid]: [30242]
[mdc.unified]: [false]
[net.bt.name]: [Android]
[net.dns1]: [210.220.163.82]
[net.dns2]: [219.250.36.130]
[net.iptype]: [601:v4]
[net.knoxvpn.version]: [2.2.0]
[net.qtaguid_enabled]: [1]
[net.smart_switch.disabled]: [1]
[net.tcp.default_init_rwnd]: [60]
[nfc.boot_reason]: [1]
[nfc.fw.dfl]: [KOR]
[nfc.fw.dfl_areacode]: [DEF]
[nfc.fw.force_download]: [0]
[nfc.fw.rfreg_display_ver]: [51]
[nfc.fw.rfreg_ver]: [20/4/24/15.42.10]
[nfc.fw.swreg_display_ver]: [51]
[nfc.fw.swreg_ver]: [20/5/26/16.44.29]
[nfc.fw.ver]: [S.LSI 6.2.6]
[nfc.initialized]: [true]
[nfc.product.support.ese]: [0]
[nfc.product.support.nonaid]: [0]
[nfc.product.support.uicc]: [1]
[nfc.product.support.uicc2]: [0]
[nfc.sent_scrs_event]: [1]
[oneseg.extSdCard.path]: [/storage/extSdCard]
[persist.audio.allsoundmute]: [0]
[persist.audio.headsetsysvolume]: [9]
[persist.audio.hphonesysvolume]: [9]
[persist.audio.pcmdump]: [0]
[persist.audio.ringermode]: [0]
[persist.audio.sales_code]: [SKC]
[persist.audio.sysvolume]: [9]
[persist.bluetooth.a2dp_offload.disabled]: [true]
[persist.bluetooth.bluetooth_audio_hal.disabled]: [true]
[persist.bluetooth.btsnoopdefaultmode]: [empty]
[persist.bluetooth.enableinbandringing]: [true]
[persist.bluetooth.scodump]: [false]
[persist.cache_integrity_64]: [2500228000]
[persist.cache_status_64]: [1]
[persist.config.chimera.enable]: [true]
[persist.data.df.agg.dl_pkt]: [10]
[persist.data.df.agg.dl_size]: [4096]
[persist.data.wda.enable]: [true]
[persist.demo.hdmirotationlock]: [false]
[persist.device_config.activity_manager_native_boot.offload_queue_enabled]: [false]
[persist.device_config.attempted_boot_count]: [0]
[persist.device_config.configuration.flag]: [false]
[persist.device_config.netd_native.dns_event_subsample_map]: [default:5 0:300 2:80 7:80]
[persist.device_config.netd_native.dot_async_handshake]: [0]
[persist.device_config.netd_native.dot_connect_timeout_ms]: [127000]
[persist.device_config.netd_native.dot_maxtries]: [3]
[persist.device_config.netd_native.keep_listening_udp]: [0]
[persist.device_config.netd_native.parallel_lookup]: [0]
[persist.device_config.netd_native.parallel_lookup_sleep_time]: [2]
[persist.device_config.netd_native.retransmission_time_interval]: [5000]
[persist.device_config.netd_native.retry_count]: [2]
[persist.device_config.netd_native.sort_nameservers]: [0]
[persist.device_config.runtime_native.usap_pool_enabled]: [true]
[persist.device_config.runtime_native.usap_pool_size_max]: [2]
[persist.device_config.runtime_native.usap_pool_size_min]: [1]
[persist.device_config.runtime_native.usap_refill_threshold]: [1]
[persist.device_config.runtime_native.use_app_image_startup_cache]: [true]
[persist.device_config.runtime_native_boot.disable_lock_profiling]: [false]
[persist.device_config.runtime_native_boot.iorap_blacklisted_packages]: []
[persist.device_config.runtime_native_boot.iorap_perfetto_enable]: [false]
[persist.device_config.runtime_native_boot.iorap_readahead_enable]: [false]
[persist.device_config.runtime_native_boot.iorapd_options]: []
[persist.device_config.storage_native_boot.fuse_enabled]: [1]
[persist.log.level]: [0xFFFFFFFF]
[persist.log.semlevel]: [0xFFFFFF00]
[persist.nfc.fw.download_onprogress]: [false]
[persist.nfc.log.index]: [4]
[persist.radio.airplane_mode_on]: [0]
[persist.radio.block_atcmd.status]: [1]
[persist.radio.latest-modeltype0]: [2]
[persist.radio.multisim.config]: [ss]
[persist.radio.sib16_support]: [1]
[persist.radio.sim.onoff]: [1]
[persist.radio.support.dualrat]: [0]
[persist.radio.tpmr_sms]: [0]
[persist.ril.config.defaultmsgapp]: [SM]
[persist.rmnet.data.enable]: [true]
[persist.rmnet.mux]: [enabled]
[persist.security.tlc.esecomm]: [1]
[persist.security.ucm_fbe_mode]: [0]
[persist.service.bdroid.version]: [5.0]
[persist.service.bgkeycount]: []
[persist.sns.camera_light]: [true]
[persist.sys.boot.reason]: []
[persist.sys.carrierid_etcpath]: [/prism/etc/carriers/LUC]
[persist.sys.ccm.date]: [Wed Apr  7 10:56:14 KST 2021]
[persist.sys.csc_status]: [normal]
[persist.sys.dalvik.vm.lib.2]: [libart.so]
[persist.sys.device_provisioned]: [1]
[persist.sys.displayinset.top]: [0]
[persist.sys.dualapp.prop]: [1]
[persist.sys.enablehomekey]: [false]
[persist.sys.fflag.override.settings_fuse]: [true]
[persist.sys.fuse]: [true]
[persist.sys.gps.lpp]: [2]
[persist.sys.isolated_storage]: [true]
[persist.sys.knox.device_owner]: [false]
[persist.sys.knox.provisioning_in_progress]: [0]
[persist.sys.knox.userinfo]: []
[persist.sys.knoxvpn.date]: [Wed Apr  7 10:56:14 KST 2021]
[persist.sys.lmk.reportkills]: [true]
[persist.sys.locale]: [ko-KR]
[persist.sys.localedefault]: [ko-KR]
[persist.sys.myfilter_preload]: [preload_done]
[persist.sys.omc.alarm]: []
[persist.sys.omc.notification]: []
[persist.sys.omc.ringtone]: []
[persist.sys.omc_etcpath]: [/prism/etc/carriers/SKC]
[persist.sys.omc_path]: [/optics/configs/carriers/SKC/conf]
[persist.sys.omc_respath]: [/prism/media/carriers/SKC]
[persist.sys.omc_root]: [/prism/etc/carriers]
[persist.sys.omc_support]: [true]
[persist.sys.omcnw_path]: [/optics/configs/carriers/SKC/conf]
[persist.sys.packagecache]: [false]
[persist.sys.prev_omcnwcode]: [SKC]
[persist.sys.prev_salescode]: [SKC]
[persist.sys.sec_cid]: [*filtered]
[persist.sys.sec_operator]: [*filtered]
[persist.sys.sec_pcid]: [*filtered]
[persist.sys.sec_subid]: [1]
[persist.sys.setupwizard]: [FINISH]
[persist.sys.sf.color_saturation]: [1.0]
[persist.sys.silent]: [1]
[persist.sys.storage_preload]: [2]
[persist.sys.theme]: [1]
[persist.sys.timezone]: [Asia/Seoul]
[persist.sys.usb.config]: [mtp,adb]
[persist.sys.vzw_wifi_running]: [false]
[persist.sys.xtra_time]: [1620796136034]
[persist.sys.zram.daily_quota_remain]: [131072]
[persist.traced.enable]: [1]
[pm.dexopt.ab-ota]: [speed-profile]
[pm.dexopt.bg-dexopt]: [speed-profile]
[pm.dexopt.boot]: [verify]
[pm.dexopt.first-boot]: [quicken]
[pm.dexopt.inactive]: [verify]
[pm.dexopt.install]: [speed-profile]
[pm.dexopt.shared]: [speed]
[restore.feature.use_samsung_smart_switch_flow]: [1]
[ril.CHAR]: [*filtered]
[ril.CompleteMsg]: [OK]
[ril.ICC_TYPE0]: [2]
[ril.LIMA]: [*filtered]
[ril.NwNmId]: [*filtered]
[ril.RildInit]: [1]
[ril.actdate]: [*filtered]
[ril.approved_codever]: [none]
[ril.approved_cscver]: [none]
[ril.approved_modemver]: [none]
[ril.atd_status]: [1_0_0]
[ril.attach.apn]: []
[ril.backoffstate]: [*filtered]
[ril.barcode]: [*filtered]
[ril.cdma.esn]: []
[ril.cold_sim]: [0]
[ril.cpreset]: [0]
[ril.cs_svc]: [1]
[ril.currentplmn]: [domestic]
[ril.data.intfprefix]: [rmnet]
[ril.dds.call.slotid]: [-1]
[ril.debug.ntc]: [*filtered]
[ril.epdg.connecting]: [-1]
[ril.epdg.currenMno]: [*filtered]
[ril.extend_band]: [true]
[ril.halservice.registered.slot1]: [true]
[ril.hasisim]: [1]
[ril.hw_ver]: [MP 0.500]
[ril.ims.ltevoicesupport]: [1]
[ril.initPB]: [1]
[ril.isIccChanged]: [0]
[ril.iscdma]: [0]
[ril.ltenetworktype]: [FDD]
[ril.manufacturedate]: [*filtered]
[ril.matchedcsc]: [SKC,KOO]
[ril.model_id]: [QB18662108]
[ril.modem.board]: [SHANNON318]
[ril.official_cscver]: [A217NOKR5CUD3]
[ril.phone.connected.slot1]: [true]
[ril.radiostate]: [10]
[ril.read.done]: [1]
[ril.region_props]: [SKC.SKC]
[ril.reject_enabled]: [1]
[ril.rejectedPlmn]: []
[ril.rfcal_date]: [*filtered]
[ril.servicestate]: [0]
[ril.signal.disprssi0]: [false]
[ril.signal.param0]: [255,14,255]
[ril.sim.acc]: [16]
[ril.sim.ef_rat_mode1]: []
[ril.sim.mobility0]: [false]
[ril.sim.opl0]: [-1]
[ril.sim.opl5g0]: [-1]
[ril.simoperator]: [ETC]
[ril.simslottype1]: [0]
[ril.simtype]: [4]
[ril.skt.network_regist]: [Status1;Idle0;CS0;PS0]
[ril.skt.network_regist_status]: [CS1;PS1]
[ril.sms_preference]: [2,0]
[ril.support.incrementalscan]: [1]
[ril.supported_rat0]: [57]
[ril.sw_ver]: [A217NKOU5CUD1]
[ril.twwan911Timer]: [0]
[ril.voice.rat]: [14]
[ril.wfc.default_spn]: [SKTelecom]
[ro.actionable_compatible_property.enabled]: [true]
[ro.adb.secure]: [1]
[ro.allow.mock.location]: [0]
[ro.apex.updatable]: [true]
[ro.arch]: [exynos850]
[ro.baseband]: [unknown]
[ro.bionic.2nd_arch]: [arm]
[ro.bionic.2nd_cpu_variant]: [cortex-a15]
[ro.bionic.arch]: [arm64]
[ro.bionic.cpu_variant]: [generic]
[ro.board.platform]: [universal3830]
[ro.boot.ap_serial]: [*filtered]
[ro.boot.asb]: [0]
[ro.boot.boot_devices]: [12100000.dwmmc0]
[ro.boot.bootloader]: [A217NKSU5CUD3]
[ro.boot.bootreason]: [reboot]
[ro.boot.carrierid]: [LUC]
[ro.boot.carrierid.param.offset]: [*filtered]
[ro.boot.cp_reserved_mem]: [off]
[ro.boot.debug_level]: [0x4f4c]
[ro.boot.dram_info]: [01,07,00,3G]
[ro.boot.dtbo_idx]: [4]
[ro.boot.dynamic_partitions]: [true]
[ro.boot.em.did]: [20018cacbffbae11]
[ro.boot.em.model]: [SM-A217N]
[ro.boot.em.rdx_dump]: [false]
[ro.boot.em.status]: [0x0]
[ro.boot.emmc_checksum]: [3]
[ro.boot.flash.locked]: [0]
[ro.boot.fmm_lock]: [48]
[ro.boot.force_upload]: [0x0]
[ro.boot.hardware]: [exynos850]
[ro.boot.hardware.sku]: [hcesim]
[ro.boot.hmac_mismatch]: [0]
[ro.boot.im.param.offset]: [7342864]
[ro.boot.me.param.offset]: [7342944]
[ro.boot.phandle]: []
[ro.boot.pr.param.offset]: [7343104]
[ro.boot.product.hardware.sku]: [hcesim]
[ro.boot.prototype.param.offset]: [7351040]
[ro.boot.recovery_offset]: [7355136]
[ro.boot.revision]: [6]
[ro.boot.rp]: [5]
[ro.boot.sales.param.offset]: [7340568]
[ro.boot.sb.debug0]: [0x0]
[ro.boot.sec_atd.tty]: [/dev/ttySAC0]
[ro.boot.selinux]: [enforcing]
[ro.boot.serialno]: [*filtered]
[ro.boot.sku.param.offset]: [7343184]
[ro.boot.sn.param.offset]: [7343024]
[ro.boot.subpcb]: [0]
[ro.boot.svb.ver]: [SVB1.0]
[ro.boot.sysup.edtbo]: [0@7381760]
[ro.boot.sysup.param]: [7361280]
[ro.boot.ucs_mode]: [0]
[ro.boot.ulcnt]: [2]
[ro.boot.verifiedbootstate]: [orange]
[ro.boot.warranty_bit]: [1]
[ro.boot.wb.hs]: [0e02]
[ro.boot.wb.snapQB]: [38909542R]
[ro.bootimage.build.date]: [Wed Apr 7 10:56:14 KST 2021]
[ro.bootimage.build.date.utc]: [1617760574]
[ro.bootimage.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.bootloader]: [A217NKSU5CUD3]
[ro.bootmode]: [unknown]
[ro.boottime.BCS-daemon]: [14551462374]
[ro.boottime.DIAG-daemon]: [14540373951]
[ro.boottime.DR-daemon]: [14544931220]
[ro.boottime.ExynosHWCConfigs]: [14157108874]
[ro.boottime.ExynosHWCServiceTW]: [14121390105]
[ro.boottime.HTduikIjCS]: [13358280182]
[ro.boottime.SMD-daemon]: [14534488682]
[ro.boottime.abox]: [14758762297]
[ro.boottime.abox_log]: [14768261412]
[ro.boottime.adbd]: [14498524066]
[ro.boottime.apexd]: [12306995645]
[ro.boottime.apexd-bootstrap]: [8292817801]
[ro.boottime.apexd-snapshotde]: [13102403490]
[ro.boottime.argos-daemon]: [14616142758]
[ro.boottime.at_distributor]: [14725426374]
[ro.boottime.atnserv]: [55820552306]
[ro.boottime.audioserver]: [14163559643]
[ro.boottime.auditd]: [13147038606]
[ro.boottime.bootanim]: [16351752373]
[ro.boottime.bootchecker]: [14701461912]
[ro.boottime.bootchecker-bootc]: [35653655782]
[ro.boottime.boringssl_self_test32]: [8975528339]
[ro.boottime.boringssl_self_test32_vendor]: [9101479801]
[ro.boottime.boringssl_self_test64]: [9040535070]
[ro.boottime.boringssl_self_test64_vendor]: [9148410378]
[ro.boottime.boringssl_self_test_apex32]: [14319523105]
[ro.boottime.boringssl_self_test_apex64]: [14373882836]
[ro.boottime.bpfloader]: [13770518644]
[ro.boottime.cameraserver]: [14509395374]
[ro.boottime.cass]: [13562151682]
[ro.boottime.cpboot-daemon]: [14623081951]
[ro.boottime.credstore]: [14190281220]
[ro.boottime.dNMYGJYXgqm4IK]: [35257268859]
[ro.boottime.derive_sdk]: [12987314413]
[ro.boottime.drk_hal]: [14068513720]
[ro.boottime.drm]: [14513825297]
[ro.boottime.dsmsd]: [14238663797]
[ro.boottime.emservice]: [13566740605]
[ro.boottime.engmode_hal]: [12059609914]
[ro.boottime.flash_recovery_sec]: [14598760220]
[ro.boottime.gatekeeperd]: [14708518220]
[ro.boottime.gpsd]: [14629596874]
[ro.boottime.gpu]: [14195567066]
[ro.boottime.health-hal-2-1-samsung]: [13990473105]
[ro.boottime.hidl_memory]: [13941561528]
[ro.boottime.hqmserv]: [23978126867]
[ro.boottime.hwservicemanager]: [10435613684]
[ro.boottime.idmap2d]: [14517897374]
[ro.boottime.imsd]: [14524571066]
[ro.boottime.incidentd]: [14529560105]
[ro.boottime.init]: [7170050071]
[ro.boottime.init.cold_boot_wait]: [1110]
[ro.boottime.init.first_stage]: [657452654]
[ro.boottime.init.fsck.cache]: [20]
[ro.boottime.init.fsck.cpefs]: [16]
[ro.boottime.init.fsck.data]: [83]
[ro.boottime.init.fsck.efs]: [16]
[ro.boottime.init.fsck.keydata]: [16]
[ro.boottime.init.fsck.keyrefuge]: [16]
[ro.boottime.init.mount.cache]: [3]
[ro.boottime.init.mount.cpefs]: [3]
[ro.boottime.init.mount.data]: [53]
[ro.boottime.init.mount.efs]: [3]
[ro.boottime.init.mount.keydata]: [2]
[ro.boottime.init.mount.keyrefuge]: [2]
[ro.boottime.init.mount_all.early]: [112]
[ro.boottime.init.mount_all.late]: [189]
[ro.boottime.init.selinux]: [233379692]
[ro.boottime.installd]: [14557079489]
[ro.boottime.insthk]: [12064640799]
[ro.boottime.iod]: [10857080492]
[ro.boottime.keystore]: [14563564220]
[ro.boottime.lmkd]: [10427449838]
[ro.boottime.logd]: [10422662646]
[ro.boottime.logd-auditctl]: [35419874282]
[ro.boottime.logd-reinit]: [13679880374]
[ro.boottime.macloader]: [14667242335]
[ro.boottime.media]: [14587696758]
[ro.boottime.media.swcodec]: [14685592874]
[ro.boottime.mediaextractor]: [14572381489]
[ro.boottime.mediametrics]: [14582021835]
[ro.boottime.multiclient-daemon]: [16432486065]
[ro.boottime.netd]: [13793430221]
[ro.boottime.network_config]: [13630183798]
[ro.boottime.nfc_hal_service]: [30243382363]
[ro.boottime.pOmHHmDQ7hneg]: [14695409566]
[ro.boottime.perfsdkserver]: [35430268244]
[ro.boottime.powersnd]: [16358969065]
[ro.boottime.prepare_param]: [14211682797]
[ro.boottime.rdxd]: [35441200744]
[ro.boottime.resetreason]: [48015563735]
[ro.boottime.ril-daemon]: [14648540758]
[ro.boottime.scs]: [11755854184]
[ro.boottime.sdp_cryptod]: [14505472720]
[ro.boottime.sec-camera-provider-4-0]: [14048868028]
[ro.boottime.sec-miscpower-2-0]: [14063147874]
[ro.boottime.sec-sysinput-1-1]: [14087197182]
[ro.boottime.sec-thermal-1-0]: [14096088836]
[ro.boottime.sec-vibrator-2-2]: [14104377951]
[ro.boottime.sec_gnss_service]: [14656229681]
[ro.boottime.servicemanager]: [10431513992]
[ro.boottime.sim_config]: [10834573838]
[ro.boottime.snap_utility_64]: [35629598128]
[ro.boottime.snapserv]: [14079217836]
[ro.boottime.statsd]: [13783113874]
[ro.boottime.storaged]: [14593973105]
[ro.boottime.storkd]: [12274748914]
[ro.boottime.surfaceflinger]: [14207528643]
[ro.boottime.system_suspend]: [12051578414]
[ro.boottime.timahidlserv]: [46496843813]
[ro.boottime.tombstoned]: [13548565067]
[ro.boottime.traced]: [14452759182]
[ro.boottime.traced_probes]: [14444287374]
[ro.boottime.tz_service]: [11764145068]
[ro.boottime.tzts_service]: [11796108876]
[ro.boottime.ueventd]: [8289439071]
[ro.boottime.usbd]: [14734157066]
[ro.boottime.vaultkeeper]: [12265909606]
[ro.boottime.vaultkeeper_hal]: [12269328914]
[ro.boottime.vendor.audio-hal]: [13946017490]
[ro.boottime.vendor.bluetooth-1-0]: [13950598490]
[ro.boottime.vendor.cas-hal-1-2]: [13955020913]
[ro.boottime.vendor.configstore-hal]: [13959505567]
[ro.boottime.vendor.drm-clearkey-hal-1-3]: [13964087528]
[ro.boottime.vendor.drm-widevine-hal-1-3]: [13968351067]
[ro.boottime.vendor.face_hal]: [14774466258]
[ro.boottime.vendor.fps_hal]: [14780478181]
[ro.boottime.vendor.gatekeeper-1-0]: [13973933836]
[ro.boottime.vendor.gralloc-2-0]: [13978745182]
[ro.boottime.vendor.hwcomposer-2-2]: [13984484374]
[ro.boottime.vendor.keymaster-4-0]: [12055893376]
[ro.boottime.vendor.media.omx]: [14610806258]
[ro.boottime.vendor.memtrack-hal-1-0]: [13995538297]
[ro.boottime.vendor.samsung.hardware.hyper-default]: [14024679067]
[ro.boottime.vendor.samsung.hardware.light-default]: [14031006682]
[ro.boottime.vendor.samsung.power-default]: [14037669528]
[ro.boottime.vendor.sensors-hal-2-0-multihal]: [14000432720]
[ro.boottime.vendor.usb-hal-1-1]: [14009451567]
[ro.boottime.vendor.wifi_hal_ext]: [14109736336]
[ro.boottime.vendor.wifi_hal_legacy]: [14019783490]
[ro.boottime.vndservicemanager]: [10439621184]
[ro.boottime.vold]: [10596724838]
[ro.boottime.wificond]: [14605376874]
[ro.boottime.wlbtd]: [14675043528]
[ro.boottime.wpa_supplicant]: [27271643557]
[ro.boottime.zygote]: [13801121221]
[ro.boottime.zygote_secondary]: [13806564721]
[ro.bt.bdaddr_path]: [/mnt/vendor/efs/bluetooth/bt_addr]
[ro.build.2ndbrand]: [true]
[ro.build.PDA]: [A217NKSU5CUD3]
[ro.build.changelist]: [21209953]
[ro.build.characteristics]: [phone]
[ro.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.build.date.utc]: [1617760574]
[ro.build.description]: [a21sks-user 11 RP1A.200720.012 A217NKSU5CUD3 release-keys]
[ro.build.display.id]: [RP1A.200720.012.A217NKSU5CUD3]
[ro.build.display_build_number]: [true]
[ro.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.build.flavor]: [a21sks-user]
[ro.build.host]: [21DJ6814]
[ro.build.id]: [RP1A.200720.012]
[ro.build.keys]: [test-keys]
[ro.build.official.release]: [true]
[ro.build.product]: [a21s]
[ro.build.selinux]: [1]
[ro.build.selinux.enforce]: [1]
[ro.build.sw_version]: [A217NKSU5CUD3]
[ro.build.system_root_image]: [false]
[ro.build.tags]: [release-keys]
[ro.build.tf_modelnumber]: [SM-A217N]
[ro.build.type]: [user]
[ro.build.user]: [dpi]
[ro.build.version.all_codenames]: [REL]
[ro.build.version.base_os]: []
[ro.build.version.codename]: [REL]
[ro.build.version.incremental]: [A217NKSU5CUD3]
[ro.build.version.min_supported_target_sdk]: [23]
[ro.build.version.preview_sdk]: [0]
[ro.build.version.preview_sdk_fingerprint]: [REL]
[ro.build.version.release]: [11]
[ro.build.version.release_or_codename]: [11]
[ro.build.version.sdk]: [30]
[ro.build.version.security_index]: [2]
[ro.build.version.security_patch]: [2021-03-01]
[ro.build.version.sem]: [3001]
[ro.build.version.sep]: [120100]
[ro.camera.notify_nfc]: [1]
[ro.carrier]: [unknown]
[ro.channel.officehubrow]: [SAMSUNG_PRELOAD]
[ro.cmc.device_type]: [pd]
[ro.cmc.version]: [2.0]
[ro.cold_boot_done]: [true]
[ro.com.google.clientidbase]: [android-samsung-ss]
[ro.com.google.clientidbase.am]: [android-lgu-kr]
[ro.com.google.gmsversion]: [11_202101]
[ro.config.alarm_alert]: [Homecoming.ogg]
[ro.config.dmverity]: [G]
[ro.config.iccc_version]: [3.0]
[ro.config.knox]: [v30]
[ro.config.knox.ucm]: [1]
[ro.config.media_sound]: [Media_preview_Touch_the_light.ogg]
[ro.config.notification_sound]: [Spaceline.ogg]
[ro.config.notification_sound_2]: [Signal.ogg]
[ro.config.pageboost.vramdisk.bootfile.enabled]: [true]
[ro.config.pageboost.vramdisk.minimize]: [true]
[ro.config.ringtone]: [Over_the_Horizon.ogg]
[ro.config.ringtone_2]: [Atomic_Bell.ogg]
[ro.config.rm_preload_enabled]: [1]
[ro.config.systemaudiodebug]: [abox&codecdsp]
[ro.config.tima]: [1]
[ro.config.timaversion]: [3.0]
[ro.config.timaversion_info]: [Knox3.4_S.R.R]
[ro.config.vc_call_vol_steps]: [8]
[ro.control_privapp_permissions]: [enforce]
[ro.crypto.state]: [unsupported]
[ro.crypto.volume.filenames_mode]: [aes-256-cts]
[ro.csc.country_code]: [KOREA]
[ro.csc.countryiso_code]: [KR]
[ro.csc.facebook.partnerid]: [samsung:dec1cc9c-1497-4aab-b953-cee702c2a481]
[ro.csc.omcnw_code]: [SKC]
[ro.csc.sales_code]: [SKC]
[ro.dalvik.vm.native.bridge]: [0]
[ro.debuggable]: [0]
[ro.em.version]: [20]
[ro.frp.pst]: [/dev/block/persistent]
[ro.gsid.image_running]: [0]
[ro.hardware]: [exynos850]
[ro.hardware.chipname]: [exynos850]
[ro.hardware.keystore]: [mdfpp]
[ro.hwui.use_vulkan]: []
[ro.iorapd.enable]: [false]
[ro.kernel.qemu]: [0]
[ro.kernel.qemu.gles]: [2]
[ro.lmk.use_minfree_levels]: [true]
[ro.logd.auditd]: [false]
[ro.logd.size.stats]: [64K]
[ro.multisim.simslotcount]: [1]
[ro.odm.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.odm.build.date.utc]: [1617760574]
[ro.odm.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.odm.build.id]: [RP1A.200720.012]
[ro.odm.build.tags]: [release-keys]
[ro.odm.build.type]: [user]
[ro.odm.build.version.incremental]: [A217NKSU5CUD3]
[ro.odm.build.version.release]: [11]
[ro.odm.build.version.release_or_codename]: [11]
[ro.odm.build.version.sdk]: [30]
[ro.odm.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.odm.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.odm.product.cpu.abilist64]: [arm64-v8a]
[ro.oem.key1]: [SKC]
[ro.oem_unlock_supported]: [1]
[ro.omc.build.id]: [39385906]
[ro.omc.build.version]: [A217NOKR5CUD3]
[ro.omc.changetype]: [DATA_RESET_ON,FALSE]
[ro.omc.disabler]: [TRUE]
[ro.omc.img_mount]: [0]
[ro.omc.region]: [KOR]
[ro.opa.eligible_device]: [true]
[ro.opengles.version]: [196610]
[ro.organization_owned]: [false]
[ro.persistent_properties.ready]: [true]
[ro.postinstall.fstab.prefix]: [/system]
[ro.product.board]: [exynos850]
[ro.product.brand]: [samsung]
[ro.product.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.product.build.date.utc]: [1617760574]
[ro.product.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.product.build.id]: [RP1A.200720.012]
[ro.product.build.tags]: [release-keys]
[ro.product.build.type]: [user]
[ro.product.build.version.incremental]: [A217NKSU5CUD3]
[ro.product.build.version.release]: [11]
[ro.product.build.version.release_or_codename]: [11]
[ro.product.build.version.sdk]: [30]
[ro.product.cpu.abi]: [arm64-v8a]
[ro.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.product.cpu.abilist64]: [arm64-v8a]
[ro.product.device]: [a21s]
[ro.product.first_api_level]: [29]
[ro.product.locale]: [ko-KR]
[ro.product.manufacturer]: [samsung]
[ro.product.model]: [SM-A217N]
[ro.product.name]: [a21sks]
[ro.product.odm.brand]: [samsung]
[ro.product.odm.device]: [a21s]
[ro.product.odm.manufacturer]: [samsung]
[ro.product.odm.model]: [SM-A217N]
[ro.product.odm.name]: [a21sks]
[ro.product.product.brand]: [samsung]
[ro.product.product.device]: [a21s]
[ro.product.product.manufacturer]: [samsung]
[ro.product.product.model]: [SM-A217N]
[ro.product.product.name]: [a21sks]
[ro.product.system.brand]: [samsung]
[ro.product.system.device]: [a21s]
[ro.product.system.manufacturer]: [samsung]
[ro.product.system.model]: [SM-A217N]
[ro.product.system.name]: [a21sks]
[ro.product.system_ext.brand]: [samsung]
[ro.product.system_ext.device]: [a21s]
[ro.product.system_ext.manufacturer]: [samsung]
[ro.product.system_ext.model]: [SM-A217N]
[ro.product.system_ext.name]: [a21sks]
[ro.product.vendor.brand]: [samsung]
[ro.product.vendor.device]: [a21s]
[ro.product.vendor.manufacturer]: [samsung]
[ro.product.vendor.model]: [SM-A217N]
[ro.product.vendor.name]: [a21sks]
[ro.product_ship]: [true]
[ro.property_service.version]: [2]
[ro.radio.noril]: [no]
[ro.revision]: [6]
[ro.ril.def_network_after_check_tdscdma]: [9]
[ro.ril.support_cdma]: [false]
[ro.ril.svdo]: [false]
[ro.ril.svlte1x]: [false]
[ro.runtime.firstboot]: [1621089963878]
[ro.secure]: [1]
[ro.security.cass.feature]: [1]
[ro.security.keystore.keytype]: [sakv2,gak]
[ro.security.reactive.version]: [2.0.12]
[ro.security.vpnpp.release]: [1.0]
[ro.security.vpnpp.ver]: [2.2]
[ro.serialno]: [RF9N8010NYL]
[ro.setupwizard.mode]: [OPTIONAL]
[ro.setupwizard.rotation_locked]: [false]
[ro.setupwizard.wifi_on_exit]: [false]
[ro.sf.lcd_density]: [280]
[ro.simbased.changetype]: [NO_DFLT_CSC,OMC]
[ro.slmk.2nd.dha_cached_max]: [14]
[ro.slmk.2nd.dha_empty_max]: [24]
[ro.slmk.2nd.freelimit_val]: [12]
[ro.slmk.2nd.swap_free_low_percentage]: [10]
[ro.slmk.3rd.dha_cached_max]: [24]
[ro.slmk.3rd.dha_cached_min]: [6]
[ro.slmk.3rd.empty_max]: [30]
[ro.slmk.3rd.over_thMB]: [4096]
[ro.slmk.add_bonusEFK]: [1]
[ro.slmk.dha_2ndprop_thMB]: [3072]
[ro.slmk.dha_cached_max]: [10]
[ro.slmk.dha_cached_min]: [3]
[ro.slmk.dha_empty_init]: [20]
[ro.slmk.dha_empty_max]: [20]
[ro.slmk.dha_pwhl_key]: [520]
[ro.slmk.freelimit_val]: [13]
[ro.slmk.plg_key]: [1]
[ro.slmk.plg_memory_th]: [2048]
[ro.slmk.psi_critical]: [150]
[ro.slmk.psi_medium]: [100]
[ro.slmk.swap_free_low_percentage]: [20]
[ro.slmk.use_lowmem_keep_except]: [true]
[ro.sts.property]: [FALSE]
[ro.surface_flinger.max_frame_buffer_acquired_buffers]: [3]
[ro.surface_flinger.running_without_sync_framework]: [false]
[ro.surface_flinger.vsync_event_phase_offset_ns]: [0]
[ro.surface_flinger.vsync_sf_event_phase_offset_ns]: [0]
[ro.sys.kernelmemory.nandswap.daily_quota]: [131072]
[ro.sys.kernelmemory.nandswap.daily_quota_limit]: [393216]
[ro.system.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.system.build.date.utc]: [1617760574]
[ro.system.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.system.build.id]: [RP1A.200720.012]
[ro.system.build.tags]: [release-keys]
[ro.system.build.type]: [user]
[ro.system.build.version.incremental]: [A217NKSU5CUD3]
[ro.system.build.version.release]: [11]
[ro.system.build.version.release_or_codename]: [11]
[ro.system.build.version.sdk]: [30]
[ro.system.build.version.sehi]: [3003]
[ro.system_ext.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.system_ext.build.date.utc]: [1617760574]
[ro.system_ext.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.system_ext.build.id]: [RP1A.200720.012]
[ro.system_ext.build.tags]: [release-keys]
[ro.system_ext.build.type]: [user]
[ro.system_ext.build.version.incremental]: [A217NKSU5CUD3]
[ro.system_ext.build.version.release]: [11]
[ro.system_ext.build.version.release_or_codename]: [11]
[ro.system_ext.build.version.sdk]: [30]
[ro.telephony.default_cdma_sub]: [0]
[ro.telephony.default_network]: [9]
[ro.telephony.iwlan_operation_mode]: [legacy]
[ro.treble.enabled]: [true]
[ro.user_trial]: [FALSE]
[ro.vendor.boot.carrierid.param.offset]: [7340596]
[ro.vendor.boot.debug_level]: [0x4f4c]
[ro.vendor.boot.force_upload]: [0x0]
[ro.vendor.boot.im.param.offset]: [7342864]
[ro.vendor.boot.me.param.offset]: [7342944]
[ro.vendor.boot.sales.param.offset]: [7340568]
[ro.vendor.boot.sn.param.offset]: [7343024]
[ro.vendor.boot.warranty_bit]: [1]
[ro.vendor.build.characteristics]: [phone]
[ro.vendor.build.date]: [Wed Apr  7 10:56:14 KST 2021]
[ro.vendor.build.date.utc]: [1617760574]
[ro.vendor.build.fingerprint]: [samsung/a21sks/a21s:11/RP1A.200720.012/A217NKSU5CUD3:user/release-keys]
[ro.vendor.build.id]: [RP1A.200720.012]
[ro.vendor.build.security_patch]: []
[ro.vendor.build.tags]: [release-keys]
[ro.vendor.build.type]: [user]
[ro.vendor.build.version.incremental]: [A217NKSU5CUD3]
[ro.vendor.build.version.release]: [11]
[ro.vendor.build.version.release_or_codename]: [11]
[ro.vendor.build.version.sdk]: [30]
[ro.vendor.config.release_version]: [20190306]
[ro.vendor.cscsupported]: [1]
[ro.vendor.epdg.support]: [true]
[ro.vendor.multisim.simslotcount]: [1]
[ro.vendor.product.cpu.abilist]: [arm64-v8a,armeabi-v7a,armeabi]
[ro.vendor.product.cpu.abilist32]: [armeabi-v7a,armeabi]
[ro.vendor.product.cpu.abilist64]: [arm64-v8a]
[ro.vendor.product_ship]: [true]
[ro.vendor.radio.default_network]: [9]
[ro.vendor.radio.default_network_by]: [telephony.prop_9]
[ro.vendor.redirect_socket_calls]: [true]
[ro.vendor.sec.radio.def_network]: [9]
[ro.vendor.security.vaultkeeper.id]: [Qq61v94394jM79gM]
[ro.vndk.version]: [30]
[ro.wifi.channels]: []
[ro.zygote]: [zygote64_32]
[secmm.codecsolution.ready]: [1]
[security.ADP.policy_version]: [20191001]
[security.ADP.version]: [3.0]
[security.ASKS.policy_version]: [20200806]
[security.ASKS.rufs_enable]: [true]
[security.ASKS.time_value]: [20210516]
[security.ASKS.version]: [4.0]
[security.perf_harden]: [1]
[security.securehw.available]: [false]
[security.securenvm.available]: [false]
[security.ucm_version]: [1.11]
[selinux.policy_version]: [SEPF_SM-A217N_11_0009]
[selinux.restorecon_recursive]: [/data/misc_ce/0]
[service.bootanim.exit]: [1]
[service.camera.running]: [0]
[service.media.powersnd]: [1]
[service.sf.present_timestamp]: [1]
[setupwizard.feature.enable_stencil_partner_customization]: [1]
[setupwizard.theme]: [glif_v3_light]
[storage.support.sdcard]: [1]
[storage.support.usb]: [1]
[sys.boot.errp]: [1]
[sys.boot.reason]: [reboot]
[sys.boot.reason.last]: [reboot]
[sys.boot_completed]: [1]
[sys.bootstat.first_boot_completed]: [1]
[sys.config.activelaunch_enable]: [true]
[sys.config.hardcoder.enable]: [false]
[sys.config.ibs.enable]: [false]
[sys.config.mars_version]: [6.00]
[sys.default.data.phoneid]: [0]
[sys.dualapp.profile_id]: []
[sys.enterprise.billing.dualsim]: [true]
[sys.enterprise.billing.version]: [1.3.0]
[sys.factory.secphone_ready]: [true]
[sys.ims.nsd.first_trigger]: [false]
[sys.is_members]: [exist]
[sys.is_vold_shutdown]: [0]
[sys.isolated_storage_snapshot]: [true]
[sys.lmk.minfree_levels]: [18432:0,23040:100,27648:200,32256:250,55296:900,80640:950]
[sys.lmk.reportkills]: [1]
[sys.members.installed]: [true]
[sys.oem_unlock_allowed]: [1]
[sys.pdp.action]: [cekeyunlock]
[sys.perf.aaal]: [true]
[sys.perf.aal]: [true]
[sys.perf.aid]: [true]
[sys.perf.bc]: [true]
[sys.perf.cg]: [true]
[sys.perf.fbooster]: [true]
[sys.perf.gvn]: [true]
[sys.perf.heavy]: [false]
[sys.perf.olaf]: [true]
[sys.perf.schd]: [true]
[sys.perf.status]: [false]
[sys.perf.tbooster]: [true]
[sys.perf.zygote]: [true]
[sys.rdxd.create]: [1]
[sys.rdxd.erase_all]: [1]
[sys.rdxd.total_size]: [4]
[sys.rescue_boot_count]: [1]
[sys.reset_reason]: []
[sys.retaildemo.enabled]: [0]
[sys.siop.level]: [-3]
[sys.smf.mnoname0]: [SKT_KR|LOADED]
[sys.sysctl.extra_free_kbytes]: [40892]
[sys.sysctl.tcp_def_init_rwnd]: [60]
[sys.sysctl.watermark_boost_factor]: [0]
[sys.system_server.start_count]: [1]
[sys.system_server.start_elapsed]: [17181]
[sys.system_server.start_uptime]: [17181]
[sys.usb.config]: [mtp,adb]
[sys.usb.configfs]: [2]
[sys.usb.configured]: [configured]
[sys.usb.controller]: [13600000.dwc3]
[sys.usb.ffs.ready]: [1]
[sys.usb.state]: [mtp,adb]
[sys.use_memfd]: [false]
[sys.user.0.ce_available]: [true]
[vendor.TEEGRIS.loaded]: [true]
[vendor.bluetooth_fw_ver]: [UTC|nacho_s612|S612_E3830_R4-1_Y21W10A]
[vendor.cbd.boot_done]: [1]
[vendor.cbd.cp_reset]: [1]
[vendor.cbd.first_xmit_done]: [1]
[vendor.cbd.rfs_check_done]: [1]
[vendor.gsm.sim.state]: [LOADED]
[vendor.gsm.version.baseband]: [A217NKOU5CUD1]
[vendor.ril.sw_ver]: [A217NKOU5CUD1]
[vendor.sec.rild.libpath]: [/vendor/lib64/libsec-ril.so]
[vendor.secureos.kernel.version]: [Samsung Secure OS Release Version 4.1.0.0 built on: 2020-11-04 11:09:29, binary version: 10]
[vendor.secureos.nwd_libs.version]: [4.1.0 Nov  4 2020 10:51:09 ]
[vendor.secureos.tzdaemon.version]: [4.1.0 compiled on: Nov  4 2020 10:51:09]
[vendor.security.vaultkeeper.native]: [1]
[vendor.tzdaemon]: [Ready]
[vendor.tzts_daemon]: [Ready]
[vendor.wifi.dualconcurrent.interface]: [swlan0]
[vendor.wifiap.defaultmaxclient]: [10]
[vendor.wifiap.wifisharinglite]: [true]
[vendor.wlan.driver.version]: [drv_ver: 11.13.0.0.0]
[vendor.wlan.firmware.version]: [O|6714|2021-03-09 20:12:01 UTC|nacho_s612|S612_E3830_R4-1_Y21W10A]
[vendor.wlbtd.board_id]: [0]
[vendor.wlbtd.tar_files_per_trigger]: [2]
[vendor.wlbtd.version]: [3.8]
[vold.has_adoptable]: [1]
[vold.has_quota]: [1]
[vold.has_reserved]: [1]
[vold.post_fs_data_done]: [1]
[wifi.active.interface]: []
[wifi.interface]: [wlan0]
[wlan.driver.status]: [unloaded]
[wlan.p2p.chkintent]: [0]
[wlan.p2p.temp]: [-3]
[wlan.p2p.wfdsta]: [disconnected]

This is the getprop I got from A217N on rooted stock UD3(R).

Reply:
@topjohnwu @osm0sis I think you should take a look at this.
There is clearly something going wrong here that you should take a look at.